{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "d62f4f83-928d-4bc8-bcd1-938b1c72a967",
   "metadata": {},
   "source": [
    "# Parse PubMed data in order to get plain text of relevant papers\n",
    "\n",
    "Through the URLs below, PubMed data can be ontained interactively. The idea here is to select papers based on keywords (and perhaps dates, ...) in a first query that results in PMC IDs. After that, the full text of those publications can be ontained one-by-one.\n",
    "\n",
    "These texts can be analyzed by scispacy language modesl that include POS tagging for biological/scienti"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "a7d9fdb1-43d8-4dd7-ac9c-b81cb566a2cf",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import pandas as pd\n",
    "import xml.etree.cElementTree as et"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "10be71cb-f2e3-40bc-9acc-87a446994541",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get data and write XML (URL should result from a query)\n",
    "response = requests.get(\"https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=p53\")\n",
    "\n",
    "data = response.content.decode()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "557a12a1-d11c-4c22-925a-a3c0a09882e3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['PMC8584119',\n",
       " 'PMC8550248',\n",
       " 'PMC8572175',\n",
       " 'PMC8571953',\n",
       " 'PMC8580964',\n",
       " 'PMC8516922',\n",
       " 'PMC8541220',\n",
       " 'PMC8583712',\n",
       " 'PMC8558325',\n",
       " 'PMC8508069',\n",
       " 'PMC8542039']"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Parse XML and extract sensible tags\n",
    "root = et.XML(data)\n",
    "\n",
    "IDs = []\n",
    "for pmc in root.iter(\"pmcid\"):\n",
    "    IDs.append(pmc.text)\n",
    "\n",
    "IDs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "2b434089-8e19-4a43-b567-1eec8f0ad106",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get full text per paper\n",
    "# process that\n",
    "# NLP that with spacy, after loading language model\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "7943ae60-aa68-4a68-a791-171736de7ed5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<italic>\n",
      "<bold>\n",
      "<sub>\n",
      "<sup>\n",
      "<label>\n"
     ]
    }
   ],
   "source": [
    "response = requests.get(\"https://www.ebi.ac.uk/europepmc/webservices/rest/PMC3257301/fullTextXML\")\n",
    "data = response.content.decode()\n",
    "\n",
    "# Perhaps remove all tags like italic, sub, label etc.\n",
    "remove = ['italic', 'bold', 'sub', 'sup', 'label']\n",
    "# Somehow, this doens't do a thing...\n",
    "for r in remove:\n",
    "    tag = '<'+r+'>'\n",
    "    print(tag)\n",
    "    data.replace(tag, '')\n",
    "    data.replace('</'+r+'>', '')\n",
    "\n",
    "# print(data)\n",
    "\n",
    "# with open(\"my_xml.xml\", 'w') as f:\n",
    "#     f.writelines(data)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "f21c830a-9c9e-4a9f-bcbe-6b497f6ac756",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Conceived and designed the experiments: FC MCL ZP FAM. Performed the experiments: FC MB WL CJL LV ZP. Analyzed the data: FC MCL MR OK ZP FAM. Contributed reagents/materials/analysis tools: MCL DS MR OK NP ZP. Wrote the paper: FC OK MCL FAM.',\n",
       " 'Like many organisms the fungal pathogen ',\n",
       " 'Skin infection, oral and vaginal thrush, or bloodstream candidiasis are some of the diseases caused by the human pathogen ',\n",
       " 'Atmospheric carbon dioxide (CO',\n",
       " 'In the fungal kingdom CO',\n",
       " 'In fungi CO',\n",
       " 'Notably the CAs of ',\n",
       " 'In this report we investigate the existence of such a pathway in fungi by using, as a model, the well characterized CO',\n",
       " 'The CA, Nce103p, from ',\n",
       " 'Inactivation of the β-carbonic anhydrase encoded by ',\n",
       " 'Since the CAs from ',\n",
       " 'We have previously shown that the adenylyl cyclase Cyr1p from ',\n",
       " 'To identify the key components of this novel CO',\n",
       " '\\n',\n",
       " 'In a previous study of transcriptional variation that follow phagocytosis of ',\n",
       " 'Since CA expression in ',\n",
       " '\\n',\n",
       " '\\n',\n",
       " 'To confirm that Rca1p directly binds to the CA promoter, and identify any additional genes it controls, we performed Chromatin Immuno Precipitation on Chip (ChIP-Chip) in air and air enriched with 5.5% CO',\n",
       " '\\n',\n",
       " 'With respect to the other Rca1p associated genes, forty four of the 85 hits were specific to ambient air samples, 19 to enriched CO',\n",
       " 'Remarkably, 46 of the 85 (54%) Rca1p associated genes are presently uncharacterized (',\n",
       " 'Since CA is critical for yeast growth in air (',\n",
       " '\\n',\n",
       " 'Our ChIP-chip data suggest a connection of Rca1p to filamentous growth and cell wall biogenesis, an observation that we confirmed by showing a strong decrease in the morphological response of ',\n",
       " 'In ',\n",
       " 'Interestingly expression levels of the Rca1p orthologues is also variable between the two species (',\n",
       " 'Sequence comparison of the Rca1p orthologs from ',\n",
       " 'Western blot with ',\n",
       " 'We have previously shown that in ',\n",
       " '\\n',\n",
       " 'Carbon dioxide is a major signal in all organisms ranging from humans to fungi ',\n",
       " 'Hydration of CO',\n",
       " 'While HCO',\n",
       " 'Though the regulation of CA by CO',\n",
       " 'Importantly, Rca1p is distinct at both the sequence and functional level from the best characterized regulator of eukaryotic CAs, HIF-1α, which induces human CA IX expression in response to hypoxia ',\n",
       " 'Similar to HIF-1α, which binds to the HRE motif present in CA IX promoter ',\n",
       " 'In ',\n",
       " 'The regulation of CA by CO',\n",
       " 'We have begun to uncover the complex physiology associated with variations in CO',\n",
       " 'The effects of CO',\n",
       " 'In conclusion, carbon dioxide is sensed in yeast by two independent pathways. One, previously described by us, involves the fungal adenylyl cyclase and cAMP ',\n",
       " 'All strains and plasmids used or constructed in this study are reported in supporting information (',\n",
       " 'Strains were inoculated in 50mL of YPD at OD',\n",
       " \"Culture and samples were prepared in an identical manner as for the protein extraction, apart from total RNA extraction which was carried out with the RNeasy Kit (Qiagen) according to the manufacturer's recommendations. Transcripts level were determined by semi quantitative RT-PCR using the iScript One-Step RT-PCR Kit with Syber Green (BioRad). Levels were normalised to \",\n",
       " '50ml cultures in YPD of strains ',\n",
       " 'Chromatin Immuno Precipitations were processed according to the above protocol with the identical strains ',\n",
       " '\\n',\n",
       " 'ScNCE103-GFP cells were observed with a Olympus IX-81 fluorescence microscope with a 150 W xenon-mercury lamp and an Olympus 60X Plan NeoFluor oil-immersion objective.',\n",
       " 'For high resolution two-photon excitation confocal microscopy of entire yeast colonies of ScNCE103-GFP, ScNCE103−GFP+',\n",
       " \"Statistical analyses were performed using Student's t test. P values are indicated as detailed in the figure legends. Error bars in figures represent SD.\",\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(TIF)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOC)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOCX)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOCX)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOCX)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOCX)',\n",
       " 'Click here for additional data file.',\n",
       " '\\n',\n",
       " '(DOCX)',\n",
       " 'Click here for additional data file.',\n",
       " 'We would like to thank Dr. C. Gourlay and Dr. T. von der Haar for their fruitful discussion and their help with ',\n",
       " 'We are grateful to the School of Biosciences, University of Kent for continuous support.',\n",
       " 'The authors have declared that no competing interests exist.',\n",
       " 'This work was funded by the BBSRC, and the MRC (all FAM), WL was the recipient of a postgraduate training grant from the Royal Thai Fellowship Program, by the Ministry of Education of the Czech Republic LC531, MSM0021620858 and AV0Z50200510 and Howard Hughes Medical Institute International Research Award to ZP. MR was funded by a grant from the Canadian Institutes of Health Research (MOP-84341), and MCL was supported by NIH award R21AI071134. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.']"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Parse XML and extract sensible tags\n",
    "root = et.XML(data)\n",
    "\n",
    "abstracts = []\n",
    "for abstract in root.iter(\"p\"):\n",
    "    abstracts.append(abstract.text)\n",
    "\n",
    "abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "39745390-effe-4687-8922-7ea018b73af4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<!DOCTYPE article PUBLIC \"-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.1 20151215//EN\" \"JATS-archivearticle1.dtd\"> \\n<article xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" article-type=\"research-article\"><?properties open_access?><?DTDIdentifier.IdentifierValue -//NLM//DTD Journal Publishing DTD v2.3 20031101//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName journalpublishing.dtd?><?SourceDTD.Version 2.3?><?ConverterInfo.XSLTName jp2nlmx2.xsl?><?ConverterInfo.Version 1?><front><journal-meta><journal-id journal-id-type=\"nlm-ta\">PLoS Pathog</journal-id><journal-id journal-id-type=\"iso-abbrev\">PLoS Pathog</journal-id><journal-id journal-id-type=\"publisher-id\">plos</journal-id><journal-id journal-id-type=\"pmc\">plospath</journal-id><journal-title-group><journal-title>PLoS Pathogens</journal-title></journal-title-group><issn pub-type=\"ppub\">1553-7366</issn><issn pub-type=\"epub\">1553-7374</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"pmcid\">3257301</article-id><article-id pub-id-type=\"pmid\">22253597</article-id><article-id pub-id-type=\"publisher-id\">PPATHOGENS-D-11-01505</article-id><article-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group><subj-group subj-group-type=\"Discipline-v2\"><subject>Biology</subject></subj-group><subj-group subj-group-type=\"Discipline-v2\"><subject>Medicine</subject></subj-group></article-categories><title-group><article-title>The bZIP Transcription Factor Rca1p Is a Central Regulator of a Novel CO<sub>2</sub> Sensing Pathway in Yeast</article-title><alt-title alt-title-type=\"running-head\">A Novel CO<sub>2</sub> Sensing Pathway in Yeast</alt-title></title-group><contrib-group><contrib contrib-type=\"author\"><name><surname>Cottier</surname><given-names>Fabien</given-names></name><xref ref-type=\"aff\" rid=\"aff1\">\\n<sup>1</sup>\\n</xref><xref ref-type=\"aff\" rid=\"aff2\">\\n<sup>2</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Raymond</surname><given-names>Martine</given-names></name><xref ref-type=\"aff\" rid=\"aff3\">\\n<sup>3</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Kurzai</surname><given-names>Oliver</given-names></name><xref ref-type=\"aff\" rid=\"aff4\">\\n<sup>4</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Bolstad</surname><given-names>Marianne</given-names></name><xref ref-type=\"aff\" rid=\"aff1\">\\n<sup>1</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Leewattanapasuk</surname><given-names>Worraanong</given-names></name><xref ref-type=\"aff\" rid=\"aff1\">\\n<sup>1</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Jim&#x000e9;nez-L&#x000f3;pez</surname><given-names>Claudia</given-names></name><xref ref-type=\"aff\" rid=\"aff5\">\\n<sup>5</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Lorenz</surname><given-names>Michael C.</given-names></name><xref ref-type=\"aff\" rid=\"aff5\">\\n<sup>5</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Sanglard</surname><given-names>Dominique</given-names></name><xref ref-type=\"aff\" rid=\"aff6\">\\n<sup>6</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>V&#x000e1;chov&#x000e1;</surname><given-names>Libu&#x00161;e</given-names></name><xref ref-type=\"aff\" rid=\"aff7\">\\n<sup>7</sup>\\n</xref><xref ref-type=\"aff\" rid=\"aff8\">\\n<sup>8</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Pavelka</surname><given-names>Norman</given-names></name><xref ref-type=\"aff\" rid=\"aff2\">\\n<sup>2</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>Palkov&#x000e1;</surname><given-names>Zdena</given-names></name><xref ref-type=\"aff\" rid=\"aff8\">\\n<sup>8</sup>\\n</xref></contrib><contrib contrib-type=\"author\"><name><surname>M&#x000fc;hlschlegel</surname><given-names>Fritz A.</given-names></name><xref ref-type=\"aff\" rid=\"aff1\">\\n<sup>1</sup>\\n</xref><xref ref-type=\"aff\" rid=\"aff9\">\\n<sup>9</sup>\\n</xref><xref ref-type=\"corresp\" rid=\"cor1\">\\n<sup>&#x0002a;</sup>\\n</xref></contrib></contrib-group><aff id=\"aff1\">\\n<label>1</label>\\n<addr-line>School of Biosciences, University of Kent, Canterbury, United Kingdom</addr-line>\\n</aff><aff id=\"aff2\">\\n<label>2</label>\\n<addr-line>Singapore Immunology Network, Agency for Science, Technology and Research, Immunos, Singapore</addr-line>\\n</aff><aff id=\"aff3\">\\n<label>3</label>\\n<addr-line>Institute for Research in Immunology and Cancer and Department of Biochemistry, Universit&#x000e9; de Montr&#x000e9;al, Montr&#x000e9;al, Quebec, Canada</addr-line>\\n</aff><aff id=\"aff4\">\\n<label>4</label>\\n<addr-line>Septomics Research Centre, Friedrich-Schiller-University and Leibniz-Institute for Natural Products Research and Infection Biology, Hans-Knoell-Institute, Jena, Germany</addr-line>\\n</aff><aff id=\"aff5\">\\n<label>5</label>\\n<addr-line>Department of Microbiology and Molecular Genetics, University of Texas Health Science Center, Houston, Texas, United States of America</addr-line>\\n</aff><aff id=\"aff6\">\\n<label>6</label>\\n<addr-line>Institute of Microbiology, University of Lausanne and University Hospital Center, Lausanne, Switzerland</addr-line>\\n</aff><aff id=\"aff7\">\\n<label>7</label>\\n<addr-line>Institute of Microbiology of the ASCR, v.v.i., Prague, Czech Republic</addr-line>\\n</aff><aff id=\"aff8\">\\n<label>8</label>\\n<addr-line>Department of Genetics and Microbiology, Faculty of Science, Charles University, Prague, Czech Republic</addr-line>\\n</aff><aff id=\"aff9\">\\n<label>9</label>\\n<addr-line>Clinical Microbiology Service, East Kent Hospitals University NHS Foundation Trust, Ashford, United Kingdom</addr-line>\\n</aff><contrib-group><contrib contrib-type=\"editor\"><name><surname>Cowen</surname><given-names>Leah E.</given-names></name><role>Editor</role><xref ref-type=\"aff\" rid=\"edit1\"/></contrib></contrib-group><aff id=\"edit1\">University of Toronto, Canada</aff><author-notes><corresp id=\"cor1\">&#x0002a; E-mail: <email>F.A.Muhlschlegel@kent.ac.uk</email></corresp><fn fn-type=\"con\"><p>Conceived and designed the experiments: FC MCL ZP FAM. Performed the experiments: FC MB WL CJL LV ZP. Analyzed the data: FC MCL MR OK ZP FAM. Contributed reagents/materials/analysis tools: MCL DS MR OK NP ZP. Wrote the paper: FC OK MCL FAM.</p></fn></author-notes><pub-date pub-type=\"collection\"><month>1</month><year>2012</year></pub-date><pub-date pub-type=\"epub\"><day>12</day><month>1</month><year>2012</year></pub-date><volume>8</volume><issue>1</issue><elocation-id>e1002485</elocation-id><history><date date-type=\"received\"><day>11</day><month>7</month><year>2011</year></date><date date-type=\"accepted\"><day>29</day><month>11</month><year>2011</year></date></history><permissions><copyright-statement>Cottier et al.</copyright-statement><copyright-year>2012</copyright-year><license xlink:href=\"http://creativecommons.org/licenses/by/4.0/\"><license-p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</license-p></license></permissions><abstract><p>Like many organisms the fungal pathogen <italic>Candida albicans</italic> senses changes in the environmental CO<sub>2</sub> concentration. This response involves two major proteins: adenylyl cyclase and carbonic anhydrase (CA). Here, we demonstrate that CA expression is tightly controlled by the availability of CO<sub>2</sub> and identify the bZIP transcription factor Rca1p as the first CO<sub>2</sub> regulator of CA expression in yeast. We show that Rca1p upregulates CA expression during contact with mammalian phagocytes and demonstrate that serine 124 is critical for Rca1p signaling, which occurs independently of adenylyl cyclase. ChIP-chip analysis and the identification of Rca1p orthologs in the model yeast <italic>Saccharomyces cerevisiae</italic> (Cst6p) point to the broad significance of this novel pathway in fungi. By using advanced microscopy we visualize for the first time the impact of CO<sub>2</sub> build-up on gene expression in entire fungal populations with an exceptional level of detail. Our results present the bZIP protein Rca1p as the first fungal regulator of carbonic anhydrase, and reveal the existence of an adenylyl cyclase independent CO<sub>2</sub> sensing pathway in yeast. Rca1p appears to regulate cellular metabolism in response to CO<sub>2</sub> availability in environments as diverse as the phagosome, yeast communities or liquid culture.</p></abstract><abstract abstract-type=\"summary\"><title>Author Summary</title><p>Skin infection, oral and vaginal thrush, or bloodstream candidiasis are some of the diseases caused by the human pathogen <italic>Candida albicans</italic>. The high versatility of infection niches reflects the capacity of this yeast to respond to strong variations in its environment such as CO<sub>2</sub> concentration. This molecule initiates the regulation of an essential protein: carbonic anhydrase, not through the known adenylyl cyclase CO<sub>2</sub> sensor but as we discovered via a novel fungal CO<sub>2</sub> sensing pathway involving the transcriptional regulator Rca1p. This protein is additionally implicated in growth, yeast-to-hyphae morphological switch and cell wall stability of <italic>C. albicans</italic>. The ortholog of Rca1p in <italic>Saccharomyces cerevisiae</italic> demonstrated a conserved function in the induction of the carbonic anhydrase in low CO<sub>2</sub> concentration atmospheres pointing to the broad significance of Rca1p in fungal CO<sub>2</sub> sensing.</p></abstract><counts><page-count count=\"13\"/></counts></article-meta></front><body><sec id=\"s1\"><title>Introduction</title><p>Atmospheric carbon dioxide (CO<sub>2</sub>) with a concentration of 0.039&#x00025; is not only central to the Earth\\'s biogeochemical carbon cycle but is also sensed as a signal by many organisms. The nematode and parasite of insects <italic>Neoaplectana carpocapsae</italic> localizes its prey via a CO<sub>2</sub> gradient <xref rid=\"ppat.1002485-Gaugler1\" ref-type=\"bibr\">[1]</xref>, while avoidance behaviour in another nematode, <italic>Caenorhabditis elegans</italic>\\n<xref rid=\"ppat.1002485-Bretscher1\" ref-type=\"bibr\">[2]</xref>, or the model organism <italic>Drosophila melanogaster</italic> is provoked by elevated CO<sub>2</sub>\\n<xref rid=\"ppat.1002485-Suh1\" ref-type=\"bibr\">[3]</xref>. <italic>C. elegans</italic> detects CO<sub>2</sub> via a cGMP-gated ion channel <xref rid=\"ppat.1002485-Bretscher1\" ref-type=\"bibr\">[2]</xref> whereas in <italic>D. melanogaster</italic> CO<sub>2</sub> is sensed by a pair of 7 transmembrane domains chemoreceptors localized on specialized sensory neurons <xref rid=\"ppat.1002485-Jones1\" ref-type=\"bibr\">[4]</xref>.</p><p>In the fungal kingdom CO<sub>2</sub>, under its hydrated form bicarbonate (HCO<sub>3</sub>\\n<sup>&#x02212;</sup>), is critical for cellular metabolism. Although hydration of CO<sub>2</sub> to HCO<sub>3</sub>\\n<sup>&#x02212;</sup> and a proton occurs spontaneously, this reaction is greatly enhanced by the metalloenzyme Carbonic Anhydrase (CA), which operates at a rate of up to 10<sup>6</sup> reactions per second <xref rid=\"ppat.1002485-Lindskog1\" ref-type=\"bibr\">[5]</xref>. Fungal CAs fix the membrane permeable gas CO<sub>2</sub> as HCO<sub>3</sub>\\n<sup>&#x02212;</sup> inside the cell, which is subsequently used as substrate for fundamental carboxylation reactions including the conversion of acetyl-CoA to malonyl-CoA (EC 6.4.1.2), or pyruvate to oxaloacetate (EC 6.4.1.1). The direct relevance of HCO<sub>3</sub>\\n<sup>&#x02212;</sup> synthesis for fungal survival is reflected by the fact that the CA deletion mutants of <italic>Candida albicans</italic>, <italic>Cryptococcus neoformans</italic>, <italic>Saccharomyces cerevisiae</italic>, <italic>Sordaria macrospora</italic>, <italic>Aspergillus fumigatus</italic> or <italic>Aspergillus nidulans</italic> fail to grow in ambient air <xref rid=\"ppat.1002485-Gotz1\" ref-type=\"bibr\">[6]</xref>, <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Elleuche1\" ref-type=\"bibr\">[8]</xref>, <xref rid=\"ppat.1002485-Han1\" ref-type=\"bibr\">[9]</xref>, <xref rid=\"ppat.1002485-Bahn1\" ref-type=\"bibr\">[10]</xref>. However, when cultured in a CO<sub>2</sub>\\n<sup>&#x02212;</sup>enriched atmosphere, where sufficient HCO<sub>3</sub>\\n<sup>&#x02212;</sup> is spontaneously formed to meet the metabolic requirements, CAs are optional.</p><p>In fungi CO<sub>2</sub> is also sensed as a signal to regulate the expression of virulence factors. In the pathogenic yeast <italic>C. albicans</italic>, high level of CO<sub>2</sub> triggers filamentous growth and the white-opaque switch <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Huang1\" ref-type=\"bibr\">[11]</xref>. Recently we have shown that in <italic>C. albicans</italic> CO<sub>2</sub>/HCO<sub>3</sub>\\n<sup>&#x02212;</sup> is detected by the enzyme adenylyl cyclase Cyr1p which regulates most processes considered essential in <italic>C. albicans</italic> virulence <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. Here, Cyr1p senses CO<sub>2</sub>/HCO<sub>3</sub>\\n<sup>&#x02212;</sup> by a lysine residue (position 1373) of the C-terminal catalytic-site <xref rid=\"ppat.1002485-Hall1\" ref-type=\"bibr\">[13]</xref> potentially linking HCO<sub>3</sub>\\n<sup>&#x02212;</sup>, generated by CA, and cAMP signaling. In humans CAs are involved in medically relevant processes including bone calcification, or renal clear-cell-carcinoma progression; consequently, understanding their regulation and use of inhibitors has attracted considerable interest <xref rid=\"ppat.1002485-Neri1\" ref-type=\"bibr\">[14]</xref>. This led to the identification of the first regulator of CA, the bHLH transcription factor HIF-1&#x003b1;, which controls the expression of major hypoxia-induced genes including CA IX <xref rid=\"ppat.1002485-Wykoff1\" ref-type=\"bibr\">[15]</xref>. Another recently identified CA regulator is AphB from <italic>Vibrio cholera</italic>\\n<xref rid=\"ppat.1002485-Kovacikova1\" ref-type=\"bibr\">[16]</xref>. This LysR-type transcription factor also activates the ToxR virulence cascade via the <italic>tcpPH</italic> operon which ultimately induces the production of cholera toxin.</p><p>Notably the CAs of <italic>S. cerevisiae</italic>\\n<xref rid=\"ppat.1002485-Aguilera1\" ref-type=\"bibr\">[17]</xref>, <xref rid=\"ppat.1002485-Amoroso1\" ref-type=\"bibr\">[18]</xref>, <italic>S. macrospora</italic>\\n<xref rid=\"ppat.1002485-Elleuche1\" ref-type=\"bibr\">[8]</xref>, <italic>A. fumigatus</italic> and <italic>A. nidulans</italic>\\n<xref rid=\"ppat.1002485-Han1\" ref-type=\"bibr\">[9]</xref> are expressed in response to the availability of environmental CO<sub>2</sub>. However, fungal genomes do not posses orthologs of either HIF-1&#x003b1; or AphB-type CA regulators. This suggests the existence of an, as yet, undiscovered CO<sub>2</sub> signaling mechanism controlling fungal CA expression.</p><p>In this report we investigate the existence of such a pathway in fungi by using, as a model, the well characterized CO<sub>2</sub> sensing system of the pathogenic yeast <italic>C. albicans</italic>. We have shown that <italic>C. albicans</italic> posses a single &#x003b2;-CA, required for growth under CO<sub>2</sub> limiting atmosphere <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>. We now demonstrate that the expression of both transcript and protein of this CA is controlled by the level of environmental CO<sub>2</sub> and that CA is further induced in an <italic>ex vivo</italic> model of phagocytosis by mammalian phagocytes, suggesting that CO<sub>2</sub> might be limiting even in the relatively high CO<sub>2</sub> conditions in the host. We find that such regulation in <italic>C. albicans</italic> is independent from the already known sensor adenylyl cyclase, described above, suggesting the existence of a cAMP-independent CO<sub>2</sub> signaling pathway in fungi. By implementing a systematic functional screen we identify the bZIP transcription factor Rca1p as the <italic>C. albicans</italic> regulator of CA expression in response to CO<sub>2</sub> availability. Furthermore, by using Chromatin Immuno Precipitation (ChIP) on chip and ChIP-qPCR experiments we confirm that Rca1p binds to the CA promoter, and to 84 additional genes. The broad significance of our findings is further underlined by our data revealing the existence of a conserved CO<sub>2</sub> sensing pathway controlled by an Rca1p ortholog in the model organism <italic>S. cerevisiae</italic>. Finally, using advanced microscopy, we contribute to the understanding of CO<sub>2</sub> flux and metabolic adaptation inside yeast populations on a hitherto unprecedented level of resolution.</p></sec><sec id=\"s2\"><title>Results</title><sec id=\"s2a\"><title>Yeast carbonic anhydrases are expressed according to CO<sub>2</sub> availability</title><p>The CA, Nce103p, from <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> are known to be required for growth in ambient air (<xref rid=\"ppat.1002485-Gotz1\" ref-type=\"bibr\">[6]</xref>, <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref> and <xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1A, B</xref>). However, to allow an in-depth analysis of yeast CA expression we developed an antibody directed against <italic>C. albicans</italic> Nce103p, and additionally constructed a strain expressing a functional tagged CA in <italic>S. cerevisiae</italic>: Sc<italic>nce103</italic>&#x00394;&#x0002b;Sc<italic>NCE103-GFP</italic> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1B</xref>).</p><fig id=\"ppat-1002485-g001\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g001</object-id><label>Figure 1</label><caption><title>\\n<italic>NCE103</italic> is essential for growth of <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> and its expression is controlled by the concentration of environmental CO<sub>2</sub>.</title><p>Inactivation of the &#x003b2;-carbonic anhydrase encoded by <italic>NCE103</italic> in (<bold>A)</bold>\\n<italic>C. albicans,</italic> (<bold>B</bold>) <italic>S. cerevisiae</italic> inhibits growth in ambient air (right set of pictures) but not in air enriched with 5.5&#x00025; CO<sub>2</sub> (left set of pictures). All strains were incubated on YPD medium for 24 hours. <bold>C</bold>) Western blots from <italic>C. albicans</italic> (left) and <italic>S. cerevisiae</italic> (right). Yeast carbonic anhydase is present in higher quantity in air than air enriched with 5.5&#x00025; CO<sub>2</sub> samples. <bold>D</bold>) qRT-PCR using <italic>NCE103</italic> specific primers and RNA extracted from <italic>C. albicans</italic> (left) and <italic>S. cerevisiae</italic> (right) grown in air (black columns) or air enriched with 5.5&#x00025; CO<sub>2</sub> (white columns). <bold>E</bold>) Western blot (top) and qRT-PCR (bottom) relative to <italic>C. albicans cyr1</italic>&#x00394;. Data are represented as mean &#x0002b;/&#x02212; SD. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p></caption><graphic xlink:href=\"ppat.1002485.g001\"/></fig><p>Since the CAs from <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> are required for growth in a low but not high CO<sub>2</sub> atmosphere we asked whether expression of the enzyme itself is regulated by the availability of this gas. To address this question we performed Western-blot analysis using appropriate antibodies to detect <italic>C. albicans</italic> CA and CA chimera of <italic>S. cerevisiae</italic>. Single bands with the predicted molecular weights for CaNce103p (32kDa), and ScNce103-GFPp (54kDa) were detected and we now show for the first time that CA protein expression is highly regulated in both yeast species (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1C</xref>). In fact, CA is strongly expressed when yeasts are grown in ambient air but non-detectable when cultured in air enriched with 5.5&#x00025; CO<sub>2</sub>, precisely mirroring the requirements for growth of the <italic>NCE103</italic> gene in yeast (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1A, B</xref>). Furthermore, quantitative real time polymerase chain reaction (qRT-PCR) analysis with reverse transcripts of total RNA extracted from <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> grown at low and high CO<sub>2</sub> concentrations show a similar regulation of CA transcript when expressions were normalized to <italic>ACT1</italic> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1D</xref>), confirming previous reports made in <italic>S. cerevisiae</italic>\\n<xref rid=\"ppat.1002485-Aguilera1\" ref-type=\"bibr\">[17]</xref>, <xref rid=\"ppat.1002485-Amoroso1\" ref-type=\"bibr\">[18]</xref>.</p></sec><sec id=\"s2b\"><title>CO<sub>2</sub> regulation of carbonic anhydrases in yeast is independent of cAMP signaling</title><p>We have previously shown that the adenylyl cyclase Cyr1p from <italic>C. albicans</italic> functions as a major CO<sub>2</sub>-sensor, promoting the yeast-to-hyphae switch in response to high levels of CO<sub>2</sub>\\n<xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Hall1\" ref-type=\"bibr\">[13]</xref>. We now asked if CO<sub>2</sub> regulation of CA expression was similarly coordinated by Cyr1p or cAMP signaling examining CA protein and transcript levels in a strain where both alleles of <italic>CYR1</italic> have been deleted (<italic>cyr1&#x00394;</italic>). Notably, CO<sub>2</sub> regulation of both protein and transcript levels of <italic>NCE103</italic> remain unaltered in the <italic>cyr1&#x00394;</italic> strain displaying a pattern of expression identical to the control strain CAI4&#x0002b;pSM2 (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1E</xref>). Furthermore, Western-blot or qRT-PCR analysis revealed that supplementation of culture media with exogenous cAMP at concentrations known to mimic Cyr1p activity <xref rid=\"ppat.1002485-Rocha1\" ref-type=\"bibr\">[19]</xref> (10 mM) did not affect CA expression in CAI4&#x0002b;pSM2 grown in ambient air (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s001\">Figure S1</xref>). Similarly to <italic>C. albicans</italic>, addition to the growth media of 10 mM cAMP did not affect the expression of CA protein or transcript in <italic>S. cerevisiae</italic> (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s001\">Figure S1</xref>). Taken together our data demonstrate that CO<sub>2</sub> regulation of CA in <italic>C. albicans</italic> is independent of the known CO<sub>2</sub> sensor Cyr1p and its product cAMP. Furthermore, they strongly support the existence of a novel, cAMP-independent, CO<sub>2</sub> signaling pathway in yeast.</p></sec><sec id=\"s2c\"><title>The bZIP transcription factor Rca1p is a new regulator of CO<sub>2</sub> signaling in <italic>C. albicans</italic>\\n</title><p>To identify the key components of this novel CO<sub>2</sub> sensing pathway we systematically screened a <italic>C. albicans</italic> knock-out library searching for strains with an altered expression of their CA in response to CO<sub>2</sub>. The library consisted of 158 <italic>C. albicans</italic> non-essential transcription factor mutants (provided by D. Sanglard). CA protein expression was investigated in each mutant grown in either ambient air or air enriched with 5.5&#x00025; CO<sub>2</sub>. Repeated screening identified a single candidate (HZY7-1) that failed to induce CA protein when grown in ambient air. HZY7-1 harbors a mutation in the <italic>C. albicans orf19.6102</italic> gene. To confirm the HZY7-1 phenotype, we independently inactivated the two <italic>orf19.6102</italic> alleles in a CAI4 background, using the URA-blaster approach <xref rid=\"ppat.1002485-Fonzi1\" ref-type=\"bibr\">[20]</xref>, and re-introduced <italic>URA3</italic> at its native locus to generate strain <italic>rca1</italic>&#x00394;. Subsequent to validating gene inactivation by Southern blot and qRT-PCR (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s002\">Figure S2</xref> and <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s003\">S3</xref>), we confirmed a striking loss of Nce103p protein induction in <italic>rca1</italic>&#x00394; in ambient air (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2A</xref>). We also validated that during the inactivation process, we did not alter the expression of 2 genes partially overlapped by <italic>RCA1</italic>: <italic>orf19.6103</italic> and <italic>MVD</italic> (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s003\">Figure S3</xref>). In light of these findings we named the gene encoded by <italic>orf19.6102</italic>: <italic><underline>R</underline>egulator of <underline>C</underline>arbonic <underline>A</underline>nhydrase <underline>1</underline>. RCA1</italic> encodes a 283 amino acid (aa) hypothetical protein which contains a conserved basic leucine zipper (bZIP) domain in its C-terminus, required for DNA interaction (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s004\">Figure S4</xref>). Reintroduction of <italic>RCA1,</italic> either on its own (<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>) or tagged at its C-terminus with Haemagglutinin (<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic>), into the <italic>rca1</italic>&#x00394; strain restored CA protein induction in <italic>C. albicans</italic> cells exposed to low CO<sub>2</sub> level (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2A</xref>). These observations were also confirmed by qRT-PCR (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2B</xref>).</p><fig id=\"ppat-1002485-g002\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g002</object-id><label>Figure 2</label><caption><title>The bZIP transcription factor Rca1p is a regulator of CO<sub>2</sub> signaling in <italic>C. albicans</italic>.</title><p>\\n<bold>A</bold>) Western blots with protein extract from the <italic>C. albicans</italic> control strain, <italic>rca1</italic>&#x00394; and <italic>rca1</italic> complemented strains. <bold>B</bold>) qRT-PCR using <italic>NCE103</italic> specific primers and RNA extracted from the above strains grown in air (black columns) or air enriched with 5.5&#x00025; CO<sub>2</sub> (white columns). <bold>C</bold>) qRT-PCR with <italic>NCE103</italic> specific primers were used to calculate the ratio of transcript between cells phagocyted and cells grown <italic>in vitro</italic> in the control (black column) or <italic>rca1</italic>&#x00394; strains (white column). Data are represented as mean &#x0002b;/&#x02212; SD. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p></caption><graphic xlink:href=\"ppat.1002485.g002\"/></fig></sec><sec id=\"s2d\"><title>\\n<italic>C. albicans NCE103</italic> induction in an <italic>ex vivo</italic> virulence model is dependent on Rca1p</title><p>In a previous study of transcriptional variation that follow phagocytosis of <italic>C. albicans</italic> by murine macrophages, <italic>NCE103</italic> was found to be mildly induced (&#x0223c;1.9-fold after two hours of co-culture <xref rid=\"ppat.1002485-Lorenz1\" ref-type=\"bibr\">[21]</xref>, while in <italic>S. cerevisiae, NCE103</italic> was one of the genes most highly induced by phagocytosis (13.8-fold; <xref rid=\"ppat.1002485-Lorenz2\" ref-type=\"bibr\">[22]</xref>). We assayed expression of <italic>CaNCE103</italic> in phagocytosed <italic>C. albicans</italic> cells after one hour of co-culture by qRT-PCR and found an induction of 2.1-fold relative to cells in media alone, even though both populations were exposed to a high-CO<sub>2</sub> environment (5.0&#x00025; in a tissue culture incubator). This change is of similar magnitude, but slightly faster, than observed by microarray. This induction was completely absent in an <italic>rca1</italic>&#x00394; strain (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2C</xref>). These results indicate that Rca1p regulates Ca<italic>NCE103</italic> in a physiological environment which could be correlated to a CO<sub>2</sub> concentration scarcer within the immune cell due to a limited penetration across multiple membranes, the sequestering activity of the mammalian CAs, or the reduced metabolic production of CO<sub>2</sub> in the fungal cell as a result of a shift to slower, and gluconeogenic growth.</p></sec><sec id=\"s2e\"><title>CO<sub>2</sub> regulation of carbonic anhydrases by Rca1p orthologs is conserved in <italic>S. cerevisiae</italic>\\n</title><p>Since CA expression in <italic>S. cerevisiae</italic> is also controlled by ambient CO<sub>2</sub> levels we investigated the existence of Rca1p orthologs in this yeast. In <italic>S. cerevisiae</italic>, we identified Cst6p (BLAST; Score: 117; E value: 1e<sup>&#x02212;16</sup>) as a potential Rca1p ortholog. Cst6p encodes for a putative 587 aa protein with a bZIP domain in the C-terminus (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s004\">Figure S4</xref>). In order to prove that Cst6p is a yeast CA regulator we constructed the mutants in the <italic>S. cerevisiae</italic> ScNCE103-GFP background (ScNCE103&#x02212;GFP&#x0002b;<italic>cst6&#x00394;</italic>). Successful gene inactivations were confirmed by diagnostic PCR and qRT-PCR (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s002\">Figure S2</xref> and <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s003\">S3</xref>). Using anti-GFP antibodies for ScNCE103&#x02212;GFP&#x0002b;<italic>cst6&#x00394;</italic> we found that its CA, similar to the <italic>C. albicans rca1&#x00394;</italic> strain (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2A</xref>), was not induced in low ambient CO<sub>2</sub> when compared to the controls (<xref ref-type=\"fig\" rid=\"ppat-1002485-g003\">Figure 3A</xref>). This regulation in <italic>S. cerevisiae</italic> mutant was also confirmed at transcript level by qRT-PCR (<xref ref-type=\"fig\" rid=\"ppat-1002485-g003\">Figure 3B</xref>). In the mutant, introduction of a plasmid expressing <italic>CST6</italic> restores the expression of <italic>NCE103</italic> in air (<xref ref-type=\"fig\" rid=\"ppat-1002485-g003\">Figure 3A</xref>). Taken together these data show that CO<sub>2</sub> regulation by Rca1p orthologs is conserved in yeast.</p><fig id=\"ppat-1002485-g003\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g003</object-id><label>Figure 3</label><caption><title>Rca1p orthologs regulate <italic>NCE103</italic> expression in <italic>S. cerevisiae</italic> via a TGACGTCA binding motif.</title><p>\\n<bold>A</bold>) Western blot with extracts from <italic>S. cerevisiae Sc</italic>NCE103&#x02212;GFP&#x0002b;pRS316 control strain, <italic>cst6</italic>&#x00394; mutant and the complemented strain (ScNCE103&#x02212;GFP&#x0002b;<italic>cst6</italic>&#x00394;&#x0002b;pRS316&#x02212;CST6). <bold>B</bold>) qRT-PCR with specific primers were used to calculate the ratio of <italic>NCE103</italic> transcript between low (air) and high CO<sub>2</sub> (5.5&#x00025;) in <italic>S. cerevisiae</italic> control strain (black column), <italic>cst6</italic>&#x00394; mutant (white column) and point mutation in the promoter of <italic>NCE103</italic> (Sc<italic>nce103&#x00394;</italic>&#x0002b;<italic>ScNCE103</italic>&#x02212;<italic>GFP</italic>&#x02212;<italic>MUT</italic>) (grey column). Data are represented as mean &#x0002b;/&#x02212; SD. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p></caption><graphic xlink:href=\"ppat.1002485.g003\"/></fig></sec><sec id=\"s2f\"><title>CO<sub>2</sub> regulation of <italic>NCE103</italic> in <italic>S. cerevisiae</italic> occurs through a specific DNA binding motif</title><p>\\n<italic>S. cerevisiae</italic> Cst6p is a transcription factor previously described to bind a specific DNA motif: TGACGTCA <xref rid=\"ppat.1002485-GarciaGimeno1\" ref-type=\"bibr\">[23]</xref>. We identified this motif in the <italic>NCE103</italic> promoters of <italic>S. cerevisiae</italic> (position -285 bp to ATG), but not of <italic>C. albicans</italic>. To assess the role of this motif in CO<sub>2</sub> regulation of CA expression we neutralized it by removing 7 and 4 bases pairs of the TGACGTCA sequence in the promoters controlling Nce103p expression in <italic>S. cerevisiae</italic>. Notably the resulting strains (Sc<italic>nce103</italic>&#x00394;&#x0002b;Sc<italic>NCE103</italic>&#x02212;<italic>GFP</italic>&#x02212;<italic>MUT</italic>) failed to induce CA when exposed to low environmental CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g003\">Figure 3B</xref>), exactly mirroring the expression pattern displayed by the <italic>S. cerevisiae cst6&#x00394;</italic> mutants (<xref ref-type=\"fig\" rid=\"ppat-1002485-g003\">Figure 3B</xref>). In summary, our data show that CO<sub>2</sub> regulation of CAs expression in yeast is controlled by a conserved transcriptional factor, but involves divergent DNA motifs between <italic>S. cerevisiae</italic> and <italic>C. albicans</italic>.</p></sec><sec id=\"s2g\"><title>ChIP-chip analysis of Rca1p confirms <italic>NCE103</italic> binding and points to a wider role in <italic>C. albicans</italic> CO<sub>2</sub> sensing</title><p>To confirm that Rca1p directly binds to the CA promoter, and identify any additional genes it controls, we performed Chromatin Immuno Precipitation on Chip (ChIP-Chip) in air and air enriched with 5.5&#x00025; CO<sub>2</sub>. We introduced the HA-tagged <italic>RCA1</italic> allele, described above, into the heterozygous <italic>RCA1</italic> mutant (<italic>rca1</italic>&#x00394;/<italic>RCA1</italic>). The resulting strain (<italic>rca1</italic>&#x00394;/<italic>RCA1</italic>&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic>) expressed one wild-type and the HA-tagged <italic>RCA1</italic> copy. Next, we confirmed that CA levels in <italic>rca1</italic>&#x00394;/<italic>RCA1</italic>&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic> and in control strain <italic>rca1</italic>&#x00394;/<italic>RCA1</italic>&#x0002b;<italic>RCA1</italic> were fully responsive to CO<sub>2</sub> by Western blotting, using anti-Nce103 antibodies (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s005\">Figure S5</xref>). Subsequently, genome-wide location profiling of Rca1-HA<sub>3</sub>p in low and high CO<sub>2</sub> using <italic>C. albicans</italic> whole-genome oligonucleotide tiling arrays <xref rid=\"ppat.1002485-Srikantha1\" ref-type=\"bibr\">[24]</xref> produced a total of 182 binding peaks, when the experiment was carried out in air, and 140 in air enriched with 5.5&#x00025; CO<sub>2</sub> (log2-transformed pseudomedian signal intensity cutoff: 0.5; P&#x02264;0.01) (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s010\">Table S1</xref> and <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s011\">S2</xref>) including 61 common &#x0201c;hits&#x0201d; between the two conditions. In depth analysis revealed that 85 of the hits could directly be associated with ORFs (hits located within 1kbp before an ATG start codon) (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4A</xref>). Notably, we identified significant enrichments of several consecutive probes localized in the promoter of <italic>NCE103</italic> (between position &#x02212;654 and &#x02212;479bp before ATG) in samples extracted from cells grown in air but not in those supplemented with 5.5&#x00025; CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4B</xref>). In addition to the promoter region, Rca1p binding was also enriched in the coding region of <italic>NCE103</italic> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4B</xref>). This binding profile has been previously reported for another bZIP transcriptional factor, Cap1p, and suggests binding of the protein to the transcriptional machinery <xref rid=\"ppat.1002485-Znaidi1\" ref-type=\"bibr\">[25]</xref>. To confirm the association of Rca1p to the CA promoter, we performed ChIP in tagged and untagged strains grown in low and high CO<sub>2</sub>, followed by a qPCR with primers specifically designed to amplify the predicted binding region of Rca1p on the <italic>NCE103</italic> promoter. As expected, we observed a 2.13 fold enrichment of this sequence in the tagged strain compared to the untagged strain in air, compared to only 1.12 fold in 5.5&#x00025; CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4C</xref>). These results show a significant association of Rca1p to the promoter of <italic>NCE103</italic> in air compared to the high CO<sub>2</sub> environment.</p><fig id=\"ppat-1002485-g004\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g004</object-id><label>Figure 4</label><caption><title>Rca1p is associated to <italic>NCE103</italic> and cell wall structure genes.</title><p>\\n<bold>A</bold>) Venn diagram of the Rca1p associated genes. <bold>B</bold>) Rca1p binds to the <italic>NCE103</italic> promoter. Representation of the normalized log<sub>2</sub>-transformed signal intensities of RCA1-HA<sub>3</sub>-tagged in air (top panel) and high CO<sub>2</sub> (bottom panel) compared to the untagged strain versus the corresponding position of each signal on <italic>C. albicans</italic> genomic regions. Log<sub>2</sub>-transformed signal intensity values are indicated at the left of the <italic>y</italic>-axis, the reference is the value 0 (i.e., a binding ratio of 1). <bold>C)</bold> ChIP-qPCR of RCA1-HA<sub>3</sub> tagged strain versus untagged control in air and a 5.5&#x00025; CO<sub>2</sub> environment normalized to <italic>ACT1</italic> level with primers designed to amplify the above identified binding region of Rca1p on the <italic>NCE103</italic> promoter. <bold>D</bold>) qRT-PCR carried out with primers for <italic>C. albicans CHT2</italic> (top) and <italic>OCH1</italic> (bottom) on total RNA extracted from the <italic>C. albicans</italic> control strain (black columns) and <italic>rca1</italic>&#x00394; (white columns) grown in air. Data are represented as mean &#x0002b;/&#x02212; SD. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p></caption><graphic xlink:href=\"ppat.1002485.g004\"/></fig><p>With respect to the other Rca1p associated genes, forty four of the 85 hits were specific to ambient air samples, 19 to enriched CO<sub>2</sub> and 22 shared between the two conditions (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4A</xref>). Rca1p binding peaks were directly associated with 4 other putative transcription factor encoding genes (<italic>CTA24</italic>, <italic>TFB3</italic>, <italic>ZCF4</italic> and <italic>ZCF22</italic>) and 2 genes involved in cell wall biosynthesis (<italic>CHT2</italic> encoding a chitinase and <italic>OCH1</italic> coding for a &#x003b1;-1,6-mannosyltransferase<italic>)</italic>. Since both <italic>CHT2</italic> and <italic>OCH1</italic> are involved in <italic>C. albicans</italic> virulence we selected them to examine the predicted role of Rca1p on their expression by qRT-PCR (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4D</xref>) <xref rid=\"ppat.1002485-Bates1\" ref-type=\"bibr\">[26]</xref>, <xref rid=\"ppat.1002485-Elson1\" ref-type=\"bibr\">[27]</xref>. Transcript levels of <italic>CHT2</italic> and <italic>OCH1</italic> were significantly higher in <italic>rca1</italic>&#x00394; in air when compared to the control strain (<xref ref-type=\"fig\" rid=\"ppat-1002485-g004\">Figure 4D</xref>). These data show that in addition to Rca1p\\'s function as activator of CA expression in low CO<sub>2</sub>, this regulator can also operate as a repressor.</p><p>Remarkably, 46 of the 85 (54&#x00025;) Rca1p associated genes are presently uncharacterized (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s010\">Table S1</xref> and <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s011\">S2</xref>). Although this observation precludes assigning a significant enrichment of genes to any cellular function, process, component (GO Term Finder, <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.candidagenome.org/cgi-bin/GO/goTermMapper\">http://www.candidagenome.org/cgi-bin/GO/goTermMapper</ext-link>) or protein families (pfam), it suggests a broader involvement of Rca1p in CO<sub>2</sub> sensing. A similar conclusion can be made following database searches with the TGACGTCA sequence involved in Cst6p binding which was retrieved in 49 promoters of <italic>S. cerevisiae</italic> genes. Analysis of both lists with GO Slim Mapper coupled to a chi-square test revealed a significant under-representation of genes in the process of RNA metabolic process (P-value: 0.0066) in <italic>C. albicans</italic> as well as in the response to chemical stimulus process for both <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> (respectively P-value 0.0436 and 0.0322) while the latter was over-represented in the budding yeast contrary to <italic>C. albicans</italic>. However, it is important to note that the number of genes involved was relatively low (respectively 2, 4 and 6). Altogether, these results show that, except for <italic>NCE103</italic>, no apparent commonality of putative Rca1p targets or pathways can be identified and the large number of uncharacterized genes in the two lists of genes poses limitations to the full elucidation of the impact of these transcriptional factors on yeast cell biology. At the same time, these data could point to the existence of yet undiscovered pathways and underline the intrinsic differences between the two fungal organisms. In summary, our data establish Rca1p as the first regulator of a fungal CA and imply a wider role of this transcription factor in a new fungal CO<sub>2</sub> sensing pathway.</p></sec><sec id=\"s2h\"><title>Rca1p regulation and role in growth, filamentation and cell wall biogenesis</title><p>Since CA is critical for yeast growth in air (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1</xref>), and its induction depends on Rca1p, it can be predicted that inactivation of <italic>RCA1</italic> should also result in a growth deficiency. Indeed we observed that <italic>rca1</italic>&#x00394; has a 77&#x00025; increase of its generation time compared to the control strain (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5A</xref>). This phenotype is not restored in high CO<sub>2</sub> pointing to a wider role of Rca1p in cell growth which could be set downstream of the CA. The enhanced growth rate of <italic>rca1</italic>&#x00394; compared to <italic>nce103</italic>&#x00394; is explained by residual expression of the highly effective carbonic anhydrase. We also confirmed that inactivation of <italic>RCA1</italic> does not lead to significant morphological alterations (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s006\">Figure S6</xref>).</p><fig id=\"ppat-1002485-g005\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g005</object-id><label>Figure 5</label><caption><title>\\n<italic>RCA1</italic> is involved in growth, cell wall structure, filamentation and is regulated by CO<sub>2</sub>.</title><p>\\n<bold>A</bold>) Generation time in YPD of <italic>C. albicans</italic> (left panel) and <italic>S</italic>. <italic>cerevisiae</italic> (right panel) control strain (black columns) and <italic>RCA1</italic> ortholog mutants (white columns) grown in air or 5.5&#x00025; CO<sub>2</sub> (grey columns). <bold>B)</bold> Germ tube formation in response to 5&#x00025; serum of <italic>C. albicans</italic> control strain (black columns) and the <italic>rca1</italic>&#x00394; (white columns) grown in air. <bold>C</bold>) Sensitivity assay of <italic>C. albicans</italic> control strain and <italic>rca1</italic>&#x00394;. <bold>D</bold>) qRT-PCR using specific primers for Ca<italic>RCA1</italic> and <italic>CST6</italic> on RNA extracted from <italic>C. albicans</italic> (top) and <italic>S. cerevisiae</italic> (bottom) control strains, CAI4&#x0002b;pSM2 and BY4741, grown in air (black columns) or air enriched with 5.5&#x00025; CO<sub>2</sub> (white columns). Data are represented as mean &#x0002b;/&#x02212; SD. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p></caption><graphic xlink:href=\"ppat.1002485.g005\"/></fig><p>Our ChIP-chip data suggest a connection of Rca1p to filamentous growth and cell wall biogenesis, an observation that we confirmed by showing a strong decrease in the morphological response of <italic>rca1</italic>&#x00394; to serum (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5B</xref> and <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s006\">S6</xref>) and an increased sensitivity of <italic>rca1</italic>&#x00394; to Congo red, caffeine and SDS (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5C</xref>). These results set Rca1p as an important player of <italic>C. albicans</italic> key biological functions.</p><p>In <italic>S. cerevisiae</italic>, we were not able to reach identical conclusions as inactivation of <italic>CST6</italic> did not result in enhanced sensitivity to cell-wall perturbing agents. Additionally, only a 20&#x00025; increase in generation time was observed for the <italic>cst6</italic>&#x00394; mutant. Notably this phenotype was complemented by growing the strain in elevated CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5A</xref>). These data confirm that the orthologs of <italic>RCA1</italic> are involved in the regulation of different cell functions further underlining their intrinsic difference emerging from the ChIP-chip and bioinformatic analysis.</p><p>Interestingly expression levels of the Rca1p orthologues is also variable between the two species (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5D</xref>). Using specific primers for each species (<italic>C. albicans</italic> and <italic>S. cerevisiae</italic>), we investigated the level of <italic>RCA1</italic> and <italic>CST6</italic> transcript in low and high CO<sub>2</sub> environment. In <italic>C. albicans</italic>, <italic>RCA1</italic> expression is 2.5 fold higher in hypercapnia compared to normal atmosphere (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5D</xref>). In contrast, the <italic>CST6</italic> transcript in <italic>S. cerevisiae</italic> did not display any significant variation of the expression between the two conditions (<xref ref-type=\"fig\" rid=\"ppat-1002485-g005\">Figure 5D</xref>). While the function as a regulator of carbonic anhydrase is shared among Rca1p orthologs, their regulation in response to environmental CO<sub>2</sub> differs.</p></sec><sec id=\"s2i\"><title>Serine 124 is involved in the regulatory function of Rca1p</title><p>Sequence comparison of the Rca1p orthologs from <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> identified three putative sites of phosphorylation (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s004\">Figure S4</xref>). We investigated the role of these residues in the function of <italic>C. albicans</italic> Rca1p by complementation of <italic>rca1</italic>&#x00394; with constructs expressing Rca1p with a replacement of serine to alanine in position 124 and 126 (<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>S124A</italic> and <italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>S126A</italic> respectively) or serine to glycine in position 222 (<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>S222G</italic>). Loss of serine in position 126 or 222 only partially impact on the CO<sub>2</sub> regulation of Nce103p expression; however mutating serine 124 lead to a striking unresponsiveness to ambient CO<sub>2</sub> resulting in enhanced expression of Nce103p in both air and air enriched with 5.5&#x00025; CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g006\">Figure 6</xref>). Our results point to a critical role of serine 124 for Rca1p activity in response to CO<sub>2</sub> concentrations.</p><fig id=\"ppat-1002485-g006\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g006</object-id><label>Figure 6</label><caption><title>Serine 124 is involved in Rca1p function as an inducer of <italic>NCE103</italic>.</title><p>Western blot with <italic>C. albicans rca1</italic>&#x00394; complemented with a wild-type allele, an <italic>RCA1</italic> allele mutated in serine 124, serine 126 or serine 222.</p></caption><graphic xlink:href=\"ppat.1002485.g006\"/></fig></sec><sec id=\"s2j\"><title>Carbonic anhydrase is differentially expressed in yeast populations</title><p>We have previously shown that in <italic>C. albicans</italic> colonies metabolically-generated CO<sub>2</sub> accumulates and is subsequently used to activate the adenylyl cyclase Cyr1p promoting the switch from yeast to filamentous growth essential for pathology <xref rid=\"ppat.1002485-Hall1\" ref-type=\"bibr\">[13]</xref>. We now substantially expand these results to entire populations of <italic>S. cerevisiae</italic> taking advantage of the regulation of expression of ScNce103-GFPp by CO<sub>2</sub>. Matching CA protein expression detected by Western blots (<xref ref-type=\"fig\" rid=\"ppat-1002485-g002\">Figure 2A</xref>), a strong fluorescent signal was recorded in ScNCE103-GFP cells grown in ambient air but absent in air enriched with 5.5&#x00025; CO<sub>2</sub> (<xref ref-type=\"fig\" rid=\"ppat-1002485-g007\">Figure 7A</xref>). Next we visualized Nce103p expression not only in individual cells but an entire fungal colony, monitoring for the first time the flux of CO<sub>2</sub> in a fungal population. Using high resolution two-photon excitation confocal microscopy <xref rid=\"ppat.1002485-Vachova1\" ref-type=\"bibr\">[28]</xref> we examined a cross-section of a ScNCE103-GFP colony grown for 4 days on solid nutrient agar. We observed that cells in the superficial layers, exposed to the low CO<sub>2</sub> concentrations found in ambient air, strongly express the Nce103-GFPp construct; while the internal layers of the colony do not show any significant fluorescence (<xref ref-type=\"fig\" rid=\"ppat-1002485-g007\">Figure 7B</xref>). Strikingly when grown in a 5.5&#x00025; CO<sub>2</sub> atmosphere, this gradient was absent, and no fluorescence was observed at any position in the colony. Similarly, no fluorescence was seen in <italic>cst6</italic>&#x00394;, regardless of the CO<sub>2</sub> concentration or the position in the colony (<xref ref-type=\"fig\" rid=\"ppat-1002485-g007\">Figure 7B</xref>), indicating that the abscence of GFP expression in the center of the ScNCE103-GFP colony grown in air was unlikely due to a lack of viability or metabolic activity of the corresponding cells. By contrast, our positive control constitutively expressing GFP displays homogenous fluorescence through the cross-section (<xref ref-type=\"fig\" rid=\"ppat-1002485-g007\">Figure 7B</xref>). In conclusion, our data visualizing the flux of CO<sub>2</sub> inside yeast populations are in full agreement with those generated by Western blot or qRT-PCR in single cells (<xref ref-type=\"fig\" rid=\"ppat-1002485-g001\">Figure 1C, D</xref>). Furthermore, they illustrate, with a high level of detail, the capacity of yeast to generate CO<sub>2</sub> enriched micro-environments and adjust metabolic expression in a population.</p><fig id=\"ppat-1002485-g007\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g007</object-id><label>Figure 7</label><caption><title>Carbonic anhydrase is differentially expressed in <italic>S. cerevisiae</italic> populations.</title><p>\\n<bold>A</bold>) Pictures of <italic>S. cerevisiae Sc</italic>NCE103-GFP cells grown in YPD for 4h in air (right panel) or air supplemented with 5&#x00025; CO<sub>2</sub> (left panel). Picture magnification x60, bar corresponds to 20 &#x000b5;m. <bold>B</bold>) Cross section of a ScNCE103-GFP colony grown in ambient air (first panel) or in air enriched with 5.5&#x00025; CO<sub>2</sub> (second panel), ScNCE103-GFP&#x0002b;<italic>cst6&#x00394;</italic> (third panel) BY4741&#x0002b;pTEF-GFP (fourth panel) were grown in air. Bar corresponds to 100 &#x000b5;m.</p></caption><graphic xlink:href=\"ppat.1002485.g007\"/></fig></sec></sec><sec id=\"s3\"><title>Discussion</title><p>Carbon dioxide is a major signal in all organisms ranging from humans to fungi <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Schmid1\" ref-type=\"bibr\">[29]</xref>. CO<sub>2</sub> regulates numerous phenotypes including virulence in the fungal pathogens of humans <italic>C. albicans</italic> or <italic>C. neoformans</italic>\\n<xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. Here we demonstrate for the first time a novel, cAMP independent, CO<sub>2</sub> sensing pathway in the yeast species <italic>C. albicans</italic> and <italic>S. cerevisiae</italic>. We report that at the core of this new sensing pathway lies a bZIP transcription factor, Rca1p in <italic>C. albicans</italic> and its ortholog Cst6p in <italic>S. cerevisiae</italic>. We show that Rca1p and its orthologs regulate the expression of a major enzyme involved in fungal metabolism, CA, in response to changes in ambient CO<sub>2</sub> level. CAs catalyze the synthesis of HCO<sub>3</sub>\\n<sup>&#x02212;</sup>, an essential substrate for the cell\\'s carboxylation reaction that sustains gluconeogenesis, ureagenesis or lipogenesis <xref rid=\"ppat.1002485-Supuran1\" ref-type=\"bibr\">[30]</xref>, <xref rid=\"ppat.1002485-Schlicker1\" ref-type=\"bibr\">[31]</xref>. We hypothesize that CA is critically involved in cellular metabolism and a feedback loop involving Rca1p could exist to regulate its expression (<xref ref-type=\"fig\" rid=\"ppat-1002485-g008\">Figure 8</xref>). Furthermore, as CA controls the level of HCO<sub>3</sub>\\n<sup>&#x02212;</sup>, the regulation of CA expression driven by cellular metabolism could have an indirect impact on the capacity of the cells to differentiate through activation of the cAMP-PKA pathway.</p><fig id=\"ppat-1002485-g008\" position=\"float\"><object-id pub-id-type=\"doi\">10.1371/journal.ppat.1002485.g008</object-id><label>Figure 8</label><caption><title>Model of CO<sub>2</sub> sensing pathways in <italic>C. albicans</italic>.</title><p>Hydration of CO<sub>2</sub> into HCO<sub>3</sub>\\n<sup>&#x02212;</sup> inside the cytoplasm occurs naturally and via the carbonic anhydrase activity. HCO<sub>3</sub>\\n<sup>&#x02212;</sup> can enter two pathways: 1) regulating morphogenesis via the activation of the adenylyl cyclase (AC) which results in an increase of cAMP production and activation of the cAMP-PKA pathway 2) integrating metabolic demand which could signal to Rca1p the cells requirement for HCO<sub>3</sub>\\n<sup>&#x02212;</sup> production resulting in the regulation of carbonic anhydrase expression controlling HCO<sub>3</sub>\\n<sup>&#x02212;</sup> synthesis rate.</p></caption><graphic xlink:href=\"ppat.1002485.g008\"/></fig><p>While HCO<sub>3</sub>\\n<sup>&#x02212;</sup> is an essential cofactor for cellular metabolism in all fungi tested, the fungal requirement for CA is conditional, depending on the environmental CO<sub>2</sub> concentration. CA mutants will not grow in ambient air where CO<sub>2</sub> is scarce but will thrive in niches where the atmosphere is enriched with this gas <xref rid=\"ppat.1002485-Gotz1\" ref-type=\"bibr\">[6]</xref>, <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Elleuche1\" ref-type=\"bibr\">[8]</xref>, <xref rid=\"ppat.1002485-Han1\" ref-type=\"bibr\">[9]</xref>, <xref rid=\"ppat.1002485-Bahn1\" ref-type=\"bibr\">[10]</xref>; the higher concentration allows sufficient spontaneous hydration to HCO<sub>3</sub>\\n<sup>&#x02212;</sup> to serve as substrate for the above carboxylation reactions. Regulation of CA expression by CO<sub>2</sub> has been reported for <italic>S. cerevisiae</italic>, <italic>S. macrospora</italic>, <italic>A. fumigatus</italic> and <italic>A. nidulans</italic>\\n<xref rid=\"ppat.1002485-Elleuche1\" ref-type=\"bibr\">[8]</xref>, <xref rid=\"ppat.1002485-Han1\" ref-type=\"bibr\">[9]</xref>, <xref rid=\"ppat.1002485-Aguilera1\" ref-type=\"bibr\">[17]</xref>, <xref rid=\"ppat.1002485-Amoroso1\" ref-type=\"bibr\">[18]</xref> and we extend these observations to <italic>C. albicans</italic>.</p><p>Though the regulation of CA by CO<sub>2</sub> has been observed in many fungi, the current work is the first to report the identification of a fungal-specific CO<sub>2</sub>-responsive transcription factor, Rca1p in <italic>C. albicans</italic> and of one of its orthologs in <italic>S. cerevisiae</italic>. We identified Rca1p, a previously uncharacterized bZIP-family DNA binding protein, via a functional genetic screen of a transcription factor knockout library. Rca1p functions by inducing CA protein and transcript when <italic>C. albicans</italic> faces low ambient CO<sub>2</sub> level. Loss of CA induction in high CO<sub>2</sub> level could result from a phosphorylation or another posttranslational modification on serine 124 which leads to the inability of Rca1p to bind <italic>NCE103</italic> promoter, integrating Rca1p in a signal transduction pathway. The impact of Rca1p in cell growth and cell wall biogenesis, independently of CO<sub>2</sub> concentration, points to a general involvement of Rca1p in the cellular metabolism of <italic>C. albicans</italic>. Rca1p function as CA regulator is conserved in <italic>S. cerevisiae</italic>, though they also have additional functions: Cst6p has already been shown to be involved in functions such as growth on non-optimal carbon sources <xref rid=\"ppat.1002485-GarciaGimeno1\" ref-type=\"bibr\">[23]</xref>, and our results now highlight the importance of Cst6p in cellular metabolism via its role as an inducer of CA. However, the impact of Cst6p on cell physiology differs when compared to <italic>C. albicans</italic> since the <italic>cts6</italic>&#x00394; mutant growth defect in air is complemented by addition of environmental CO<sub>2</sub> for <italic>S. cerevisiae</italic>. Furthermore the observation that <italic>RCA1</italic> expression itself is regulated by CO<sub>2</sub> underlines the importance of this regulator outside the scope of CA expression. Notably, Rca1p orthologs can be identified in <italic>S. macrospora</italic>, <italic>A. fumigatus</italic> and <italic>A. nidulans</italic> known to posses CA\\'s which expression is influenced by ambient CO<sub>2</sub> level <xref rid=\"ppat.1002485-Elleuche1\" ref-type=\"bibr\">[8]</xref>, <xref rid=\"ppat.1002485-Han1\" ref-type=\"bibr\">[9]</xref>.</p><p>Importantly, Rca1p is distinct at both the sequence and functional level from the best characterized regulator of eukaryotic CAs, HIF-1&#x003b1;, which induces human CA IX expression in response to hypoxia <xref rid=\"ppat.1002485-Wykoff1\" ref-type=\"bibr\">[15]</xref>, <xref rid=\"ppat.1002485-Schofield1\" ref-type=\"bibr\">[32]</xref>. CA IX leads to extracellular acidification of hypoxic tissue, and is as such abundant in tumors <xref rid=\"ppat.1002485-Parks1\" ref-type=\"bibr\">[33]</xref>. By contrast <italic>C. albicans NCE103</italic>, is not regulated by either hypoxia or changes in pH (<xref rid=\"ppat.1002485-Setiadi1\" ref-type=\"bibr\">[34]</xref>; <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s007\">Figure S7</xref>).</p><p>Similar to HIF-1&#x003b1;, which binds to the HRE motif present in CA IX promoter <xref rid=\"ppat.1002485-Schofield1\" ref-type=\"bibr\">[32]</xref>, the regulation of <italic>NCE103</italic> by Rca1p appears to be direct. ChIP-chip and ChIP-qPCR confirmed that Rca1p binds to the CA promoter of <italic>C. albicans,</italic> specifically under low ambient CO<sub>2</sub>, leading to the induction of <italic>NCE103</italic> expression. An additional 84 genes were associated with Rca1p, suggesting a much broader role of this new CO<sub>2</sub> signaling pathway. Although the majority of these genes are currently of unknown function, in depth analysis of two (<italic>OCH1</italic> and <italic>CHT2</italic>) showed Rca1p\\'s potential to act as both an inducer and repressor of gene expression. This dual function could also be explained by an ability of Rca1p to recruit different co-factors at the associated loci. Using SCOPE <xref rid=\"ppat.1002485-Carlson1\" ref-type=\"bibr\">[35]</xref> and other predictive programs, we obtained a relatively low number of results with significant value regarding binding sites or processes associated to Rca1p and Cst6p. These observations could be due to a higher complexity in the binding motif of Rca1p to the DNA, as well as on our limited knowledge about genes function (75&#x00025; of ORF in <italic>C. albicans</italic> are still considered as uncharacterized; <ext-link ext-link-type=\"uri\" xlink:href=\"http://www.candidagenome.org\">http://www.candidagenome.org</ext-link>). Reflecting the 235 million years <xref rid=\"ppat.1002485-Taylor1\" ref-type=\"bibr\">[36]</xref> of separation between the <italic>Candida</italic> and <italic>Saccharomyces</italic> clade, we observed profound divergence in the associated genes of Rca1p and Cst6p. However, the CA remains a conserved target for both species. Such a fundamental wiring re-arrangement between closely related transcriptional factor of <italic>C. albicans</italic> and <italic>S. cerevisiae</italic> has already been reported <xref rid=\"ppat.1002485-Lavoie1\" ref-type=\"bibr\">[37]</xref>.</p><p>In <italic>S. cerevisiae</italic>, CO<sub>2</sub> regulation of CA expression by Cst6p involves the TGACGTCA palindrome motif. Database searches with this sequence retrieved the motif in 49 promoter regions of <italic>S. cerevisiae</italic>, and 40 promoters of <italic>C. albicans</italic>. While this motif is absent in the <italic>NCE103</italic> promoter in <italic>C. albicans</italic>, it is present in a single gene associated to Rca1p (<italic>orf19.4246</italic>) demonstrating the lack of motifs conservation between the two species. Interestingly<underline>,</underline> the promoter of human CA IX presents a bZIP binding motif, TGAGTCA <xref rid=\"ppat.1002485-Kaluz1\" ref-type=\"bibr\">[38]</xref>, which is closely related to the one identified in <italic>S. cerevisiae</italic>. Furthermore this motif is the binding sequence of the oncogen C-jun in human, which presents some sequence similarity with Rca1p (Score 42.7, e value 0.003), particularly around the bZIP domain. To date, the expression of CA IX in response to CO<sub>2</sub> changes in the body has not been investigated. Our results may point to an additional level of regulation in human CA\\'s; in fact HIF-1&#x003b1; sole predominance in CA IX regulation has recently been questioned <xref rid=\"ppat.1002485-Brennan1\" ref-type=\"bibr\">[39]</xref>.</p><p>The regulation of CA by CO<sub>2</sub> is likely to be complex; however, using a new antibody that we generated, we show that CO<sub>2</sub> affects CA proteins levels dramatically &#x02013; highly induced in normal and undetectable when cells are grown in an elevated CO<sub>2</sub> atmosphere. However, CA transcript levels were only decreased by 50&#x00025; relative to ambient air. This type of regulation may suggest additional levels of post-transcriptional control on CA messenger. Maintaining CA transcripts in high CO<sub>2</sub> level would allow a shortened response in CA enzyme synthesis when cells encounter a switch from high to low CO<sub>2</sub> atmosphere, thus ensuring sufficient supply of the essential HCO<sub>3</sub>\\n<sup>&#x02212;</sup> ion.</p><p>We have begun to uncover the complex physiology associated with variations in CO<sub>2</sub> concentrations. <italic>C. albicans</italic> cells phagocytosed by macrophages induce CA, as seen by qRT-PCR, despite being in a high CO<sub>2</sub> (5&#x00025;) environment, and this is Rca1p-dependent. This suggests that the phagocyte might restrict CO<sub>2</sub> availability, as it does for other nutrients. Furthermore, using high resolution two-photon excitation confocal microscopy with GFP-tagged CA\\'s we visualize for the first time the impact of CO<sub>2</sub> build-up on gene expression in single cells but also in entire fungal populations. The data presented in this report not only confirm previous observations made in <italic>C. albicans</italic> that CO<sub>2</sub> is compartmentalized in yeast populations, specifically inducing developmental change <xref rid=\"ppat.1002485-Hall1\" ref-type=\"bibr\">[13]</xref>, but importantly connects micro-environments enriched in CO<sub>2</sub> to metabolic specialization of individual members of a fungal populations. However, it is important to also consider the possibility that an unknown, CO<sub>2</sub>-independent, pathway is involved in the regulation of Nce103p at colony level. CA regulation is exquisitely sensitive to change in CO<sub>2</sub> availability in both <italic>S. cerevisiae</italic> and <italic>C. albicans</italic> (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s008\">Figure S8</xref>) and our results may be highly applicable to a range of conditions in which fungi expand and act as populations rather than individual cells including the formation of drug-resistant biofilms in pathogenic yeasts such as <italic>C. albicans</italic>. For <italic>S. cerevisiae</italic>, biofilms are naturally isolated on fruit surface (grape), and have a major application in industrial fermentation <xref rid=\"ppat.1002485-Demirci1\" ref-type=\"bibr\">[40]</xref>, <xref rid=\"ppat.1002485-Zara1\" ref-type=\"bibr\">[41]</xref>.</p><p>The effects of CO<sub>2</sub> on fungal physiology are integrated through more than one regulatory circuit. We previously showed that elevated CO<sub>2</sub>/HCO<sub>3</sub>\\n<sup>&#x02212;</sup> is sensed by the adenylyl cyclase via a lysine residue of the enzymes catalytic core, increasing the production of the second messenger cAMP, thus linking adenylyl cyclase and CA in fungal CO<sub>2</sub> sensing <xref rid=\"ppat.1002485-Klengel1\" ref-type=\"bibr\">[7]</xref>, <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. Adenylyl cyclase/cAMP are particularly important mediators of fungal virulence determinants. Although there is cross-talk between the activities of the two enzymes, we now show that the CO<sub>2</sub> control of CA expression acts independently from the cAMP-PKA pathway. This mechanism has to be compared to the identification of a cAMP-independent CO<sub>2</sub>-sensitive pathway involved in white opaque switching which results in Wor1p phosphorylation <xref rid=\"ppat.1002485-Huang2\" ref-type=\"bibr\">[42]</xref>. However, the putative overlap of these two uncharacterized pathways remains to be defined.</p><p>In conclusion, carbon dioxide is sensed in yeast by two independent pathways. One, previously described by us, involves the fungal adenylyl cyclase and cAMP <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. We have now identified a second pathway and found the transcriptional regulator, Rca1p, at the core of the new pathway in yeast. Activation or inactivation of the transcription factor may involve phosphorylation which ultimately programs cellular metabolism to allow optimal adaptation to the environment inside a macrophage in the case of a fungal pathogen or yeast populations. Investigating the function of the additional Rca1p-associated genes will bring better understanding on how organisms sense the universal gas carbon dioxide.</p></sec><sec sec-type=\"materials|methods\" id=\"s4\"><title>Materials and Methods</title><sec id=\"s4a\"><title>Strains</title><p>All strains and plasmids used or constructed in this study are reported in supporting information (<xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s012\">Table S3</xref>, <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s013\">S4</xref>, <xref ref-type=\"supplementary-material\" rid=\"ppat.1002485.s014\">S5</xref>), as is the composition of the respective media and additional protocols (Protocol S1). <italic>C. albicans</italic> were incubated at 37&#x000b0;C, or 30&#x000b0;C for <italic>S. cerevisiae</italic>, either in ambient air or air enriched with 5.5&#x00025; (vol/vol) atmospheric CO<sub>2</sub> (Infors HT Minitron) when required.</p></sec><sec id=\"s4b\"><title>Protein extraction and Western blot</title><p>Strains were inoculated in 50mL of YPD at OD<sub>600</sub> 0.1 and grown at the suitable temperature in air or air enriched with CO<sub>2</sub>. After 4h, cells were collected and quickly frozen. Samples were disrupted using a Mikro-dismembrator S (Sartorius) and resuspended in 500 &#x000b5;l lysis buffer (50 mM HEPES, 150 mM NaCl, 5 mM EDTA, 1&#x00025; Triton X-100, protease inhibitor [Roche]). Protein concentrations were quantified using Bradford reagent (Sigma). 30 &#x000b5;g of protein were loaded for each sample on a 12&#x00025; SDS-acrylamide/bis-acrylamide gel, and proteins were transferred to a PVDF membrane (Millipore). Membranes were incubated with the appropriate antibodies diluted as follows: anti-NCE103 at 1&#x02236;500, anti-LacZ at 1&#x02236;1000 (Millipore), anti-GFP at 1&#x02236;2500 (Roche) or anti-Act1p 1&#x02236;1000 (Sigma). This was followed by a second incubation with a peroxidase tagged antibodies of goat anti-rabbit, diluted 1&#x02236;2000, (Sigma) for the anti-NCE103, anti-LacZ and anti-Act1 primary antibodies, while a goat anti-mouse, diluted 1&#x02236;5000 (Sigma), was used for the anti-GFP primary antibody. Luminol Electrochemiluminescence was used to detect signal on the membrane.</p></sec><sec id=\"s4c\"><title>qRT-PCR</title><p>Culture and samples were prepared in an identical manner as for the protein extraction, apart from total RNA extraction which was carried out with the RNeasy Kit (Qiagen) according to the manufacturer\\'s recommendations. Transcripts level were determined by semi quantitative RT-PCR using the iScript One-Step RT-PCR Kit with Syber Green (BioRad). Levels were normalised to <italic>ACT1</italic> from the respective species, and calculated using the Delta C(t) analysis of the Opticon Monitor 3 software (Bio-Rad). Values are represented as mean &#x0002b;/&#x02212; SD from three independent experiments.</p></sec><sec id=\"s4d\"><title>Chromatin Immuno Precipitation on Chip</title><p>50ml cultures in YPD of strains <italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic> (untagged) and <italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic> (tagged) were inoculated at OD<sub>600</sub> 0.1 by overnight culture and incubated 4h at 37&#x000b0;C in air or air enriched with 5.5&#x00025; CO<sub>2</sub>, 140 rpm. Three independents cultures were grown for each strain in both conditions. The subsequent steps of DNA cross-linking, DNA shearing, chromatin immunoprecipitation (ChIP), DNA labeling with Cy3 or Cy5 dyes, hybridization to intergenic DNA microarrays, and data analysis were conducted exactly as described <xref rid=\"ppat.1002485-Huang2\" ref-type=\"bibr\">[42]</xref>. Cy5-labeled DNA from the tagged strain (<italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic>) and the corresponding Cy3-labeled DNA from the untagged control strain (<italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>) were mixed and hybridized to a <italic>C. albicans</italic> whole-genome tiled oligonucleotide DNA microarray <xref rid=\"ppat.1002485-Srikantha1\" ref-type=\"bibr\">[24]</xref>. After hybridization and scanning of the slides (<italic>n</italic>&#x0200a;=&#x0200a;3 for each condition), results were process <xref rid=\"ppat.1002485-Liu1\" ref-type=\"bibr\">[43]</xref>. Quantile normalization was applied to the data <xref rid=\"ppat.1002485-Znaidi1\" ref-type=\"bibr\">[25]</xref>. The parameters used were: a window size of 400 bp, a maximum genomic distance of 60 bp, and a minimum length of 120 bp. The replicate data were combined, and peak finding (i.e., determining the Rca1-HA<sub>3</sub>p binding sites) was done using a pseudomedian signal threshold of at least 1.5 fold and a <italic>P</italic> value cutoff of 0.01 <xref rid=\"ppat.1002485-Znaidi1\" ref-type=\"bibr\">[25]</xref>, <xref rid=\"ppat.1002485-Zhang1\" ref-type=\"bibr\">[44]</xref>.</p></sec><sec id=\"s4e\"><title>ChIP-qPCR</title><p>Chromatin Immuno Precipitations were processed according to the above protocol with the identical strains <italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic> (untagged) and <italic>RCA1</italic>/<italic>rca1</italic>&#x00394;&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic> (tagged) and growth conditions. The resulting purified DNA was used in quantitative PCR using SYBR Green Master Mix (Applied Biosystems, Inc.) with primers: Ca-ChIP-NCE103-F/Ca-ChIP-NCE103-R for the detection of the <italic>NCE103</italic> promoter (a 195bp region identified to be significantly associated with Rca1p) and Ca-ChIP-ACT1-F/Ca-ChIP-ACT1-R for the control <italic>ACT1</italic> promoter, a gene without known association for Rca1p. Levels of detection were normalized to <italic>ACT1</italic> and calculated using the Delta Delta C(t) method. Values are represented as mean &#x0002b;/&#x02212; SD from two independent experiments.</p></sec><sec id=\"s4f\"><title>Macrophage co-culture experiments</title><p>\\n<italic>C. albicans</italic> wild-type (SC5314) and <italic>rca1&#x00394;</italic> were grown to log-phase in YPD medium, washed in water, and counted. They were then incubated with J774A.1 macrophages at an MOI of 2&#x02236;1 (<italic>C. albicans</italic>: macrophages) in RPMI&#x0002b;10&#x00025; FBS at 37&#x000b0;C in 5&#x00025; CO<sub>2</sub> in 750 cm<sup>3</sup> vented flasks. Control cells were grown in the same media without macrophages at 37&#x000b0;C in 5&#x00025; CO<sub>2</sub>. After incubation for one hour, the flasks were rinsed with PBS, then cells were collected by scraping into ice cold water and transferred to conical tubes. They were washed twice more with water, then pellets were frozen on dry ice. RNA was prepared using the Turbo DNA-free kit (Ambion). 50 ng of total RNA were used for each qRT-PCR reaction using the Power SYBR Green reaction system (Invitrogen). Actin (<italic>ACT1</italic>) was used as the normalization control. Primers are listed in supplemental data.</p></sec><sec id=\"s4g\"><title>Microscopy</title><p>ScNCE103-GFP cells were observed with a Olympus IX-81 fluorescence microscope with a 150 W xenon-mercury lamp and an Olympus 60X Plan NeoFluor oil-immersion objective.</p><p>For high resolution two-photon excitation confocal microscopy of entire yeast colonies of ScNCE103-GFP, ScNCE103&#x02212;GFP&#x0002b;<italic>cst6</italic>&#x00394; and BY4741&#x0002b;pTEF&#x02212;GFP, cells were grown for 4 days on YPD at 28&#x000b0;C. Colonies were then embedded in low-gelling agarose (Sigma-Aldrich) directly on the plates <xref rid=\"ppat.1002485-Vachova1\" ref-type=\"bibr\">[28]</xref>. After solidification, agarose-embedded colonies (an area of approximately 10&#x000d7;10 mm) were sectioned vertically down the middle and transferred to the cover glass. All samples on the cover slip were enclosed with a thick agarose layer to prevent them from drying. Image acquisition was realized following published protocol <xref rid=\"ppat.1002485-Vachova1\" ref-type=\"bibr\">[28]</xref>, using 20 x/0.7 water immersion planachromat objective.</p></sec><sec id=\"s4h\"><title>Statistical analysis</title><p>Statistical analyses were performed using Student\\'s t test. P values are indicated as detailed in the figure legends. Error bars in figures represent SD.</p></sec></sec><sec sec-type=\"supplementary-material\" id=\"s5\"><title>Supporting Information</title><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s001\"><label>Figure S1</label><caption><p>\\n<bold>Carbonic anhydrase expression is independent of the cAMP-PKA pathway.</bold> Using our anti-Nce103p and anti-GFP antibodies, carbonic anhydrase signals are shown in western blots from <italic>C. albicans</italic> (top) and <italic>S. cerevisiae</italic> (bottom). Proteins were extracted from cells grown in YPD for 4h in air (with or without addition of 10 mM dbcAMP to the culture medium) or air enriched with 5.5&#x00025; CO<sub>2</sub>. Yeast carbonic anhydrase expression is not influenced by the addition of dbcAMP. The same samples were probed with an anti-actin antibody as control.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s001.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s002\"><label>Figure S2</label><caption><p>\\n<bold>Strains verification in </bold>\\n<bold><italic>C. albicans</italic></bold>\\n<bold> and </bold>\\n<bold><italic>S. cerevisiae</italic></bold>\\n<bold>. A</bold>) Southern blot where genomic DNA from strain CAI4&#x0002b;pSM2 (1), <italic>rca1&#x00394;</italic>&#x0002b;<italic>RCA1</italic> (2), <italic>rca1&#x00394;</italic>&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic> (3), <italic>rca1&#x00394;</italic> (4), <italic>rca1&#x00394;</italic>/<italic>RCA1</italic>&#x0002b;<italic>RCA1</italic> (5 and 6) and <italic>rca1&#x00394;</italic>/<italic>RCA1</italic>&#x0002b;<italic>RCA1</italic>&#x02212;<italic>HA<sub>3</sub></italic> (7) were digested by <italic>Sac</italic>I, migrated on agarose gel and transfered onto nitrocellulose membrane. Using a <italic>RCA1</italic> probe, expected bands were observed with a signal at 3.8kbp for the <italic>RCA1</italic> allele, 4.6kbp for <italic>rca1&#x00394;</italic>, 8.1 and 8 kbp for the introduction of the pSM2-RCA1 and pSM2-RCA1-HA<sub>3</sub> alleles respectively. <bold>B</bold>) Diagnostic PCR products with primers NCE103-Verif-F and ScNCE-end using genomic <italic>S. cerevisiae</italic> DNA of control strain BY4741 (lane 1) and Sc<italic>nce103&#x00394;</italic> (lane 3) as template, or primers Nce.ko.kan-F and NCE103-Verif-R with the respective template on lane 2 and 4. First set of primers hybridize on each side of the cassette, second set confirm presence of the cassette at the right locus. <bold>C</bold>) Diagnostic PCR products with primers CST6-Verif-F and CST6-Verif-R using genomic <italic>S. cerevisiae</italic> DNA of control strain ScNCE103-GFP (lane 1) and ScNCE103-GFP&#x0002b;<italic>cst6&#x00394;</italic> (lane 3) as template, or primers ScCST6.ko.kan-F and CST6-Verif-R with the respective template on lane 2 and 4. First set of primers hybridize on each side of the cassette, second set confirm presence of the cassette at the right locus.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s002.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s003\"><label>Figure S3</label><caption><p>\\n<bold>Verification of gene expression by qRT-PCR. A</bold>) qRT-PCR using <italic>RCA1</italic> specific primers and RNA extracted from <italic>C. albicans</italic> control strain and the <italic>RCA1</italic> mutant grown in air. The <italic>rca1&#x00394;</italic> strains show no significant level of <italic>RCA1</italic> transcript. <bold>B</bold>) qRT-PCR using Sc<italic>NCE103</italic> (top) and <italic>CST6</italic> (bottom) specific primers and RNA extracted from the <italic>S. cerevisiae</italic> controls, Sc<italic>NCE103</italic> mutant and <italic>CST6</italic> mutant strain grown in air enriched with 5.5&#x00025; CO<sub>2</sub> (top panel) or air (bottom panel). Both mutants show no significant level of Sc<italic>NCE103</italic> and <italic>CST6</italic> transcript compared to the control strain. <bold>C</bold>) qRT-PCR using <italic>ORF19.6103</italic> (top) and <italic>MVD</italic> (bottom) specific primers and RNA extracted from <italic>C. albicans</italic> control and the <italic>RCA1</italic> mutant grown in air (black columns) or air enriched with 5.5&#x00025; CO<sub>2</sub> (white columns). Expression of both genes is not significantly different between the control and mutant strain. Data are represented as mean &#x0002b;/&#x02212; SD from three independent experiments. Asterisk indicates statistical significance determined by two-sample <italic>t</italic> test (<italic>P</italic>&#x02264;0.05).</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s003.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s004\"><label>Figure S4</label><caption><p>\\n<bold>Protein alignment of Rca1p and Cst6p sequences.</bold> Alignment of <italic>C. albicans</italic> Rca1p (C.a.) and <italic>S. cerevisiae</italic> Cst6p (S.c.) sequences by ClustalW2 (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.ebi.ac.uk/Tools/clustalw2/index.html\">http://www.ebi.ac.uk/Tools/clustalw2/index.html</ext-link>). &#x0201c;&#x0002a;&#x0201d;, &#x0201c;:&#x0201d; and &#x0201c;.&#x0201d; respectively means that the residues of that column are identical in the two sequences, that conserved substitutions occurred, or that semi-conserved substitutions are observed. The bZIP motifs (bold) are present in the C-terminus of each protein. 3 conserved putative serine sites for phosphorylation (underlined) are shown (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.cbs.dtu.dk/services/NetPhosYeast/\">http://www.cbs.dtu.dk/services/NetPhosYeast/</ext-link>).</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s004.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s005\"><label>Figure S5</label><caption><p>\\n<bold>The </bold>\\n<bold><italic>RCA1</italic></bold>\\n<bold> heterozygous mutant and complemented strains display a wild-type pattern of Nce103p expression.</bold> Carbonic anhydrase signals are shown in western blots from the <italic>C. albicans</italic> control, <italic>RCA1</italic> heterozygous mutant, and the complemented strains. All strains display an identical profile of Nce103p expression. The same samples were probed with an anti-actin antibody as control.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s005.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s006\"><label>Figure S6</label><caption><p>\\n<bold><italic>RCA1</italic></bold>\\n<bold> inactivation does not impact on cells morphology.</bold> Control (CAI4&#x0002b;pSM2) and <italic>rca1</italic> mutant (<italic>rca1</italic>&#x00394;) strain were grown for 2h at 37&#x000b0;C in YPD supplemented or not with 5&#x00025; horse serum. Representative pictures show identical morphology for both strains in YPD and confirm the inability of the <italic>rca1</italic> mutant to differentiate into hyphae. Bar corresponds to 5 &#x000b5;m.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s006.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s007\"><label>Figure S7</label><caption><p>\\n<bold><italic>C.albicans</italic></bold>\\n<bold> Nce103p is not regulated by environmental pH.</bold> Western blots showing carbonic anhydrase signals from the <italic>C. albicans</italic> control strain grown in YPD buffered at pH 4 or pH 7 for 4h in air. In both conditions, Nce103p signals are identical.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s007.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s008\"><label>Figure S8</label><caption><p>\\n<bold>Nce103p induction is exquisitely sensitive to ambient CO<sub>2</sub> availability.</bold> Using appropriate antibodies, carbonic anhydrase signals are shown in western blots from <italic>S. cerevisiae</italic> (top) and <italic>C. albicans</italic> (bottom). Proteins were extracted from cells grown in YPD for 4h in air or air enriched with 0.5 or 1&#x00025; CO<sub>2</sub>. Nce103p signals are detectable in ambient air, and air enriched with 0.5&#x00025; CO<sub>2</sub>.</p><p>(TIF)</p></caption><media xlink:href=\"ppat.1002485.s008.tif\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s009\"><label>Protocol S1</label><caption><p>\\n<bold>Detailed protocols about media used in this study, strains and plasmids construction, yeast transformations, Southern blot analysis and the generation of </bold>\\n<bold><italic>C. albicans</italic></bold>\\n<bold> Nce103p antibodies, as well as supporting references.</bold>\\n</p><p>(DOC)</p></caption><media xlink:href=\"ppat.1002485.s009.doc\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s010\"><label>Table S1</label><caption><p>\\n<bold>Rca1p-HA<sub>3</sub> binding in air dataset.</bold> The following criteria were used: Log<sub>2</sub> pseudo-median signal intensity threshold of &#x02265;0.5 and p-value cut-off of &#x02264;0.01 <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. <bold>Contig19#:</bold> The Contig19 number on which a given binding peak is detected using the Tilescope software <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. <bold>Location:</bold> Position of the binding peak in the corresponding Contig19 DNA sequence. <bold>Log2 pseudo-median signal intensity:</bold> Log<sub>2</sub>-transformed pseudo-median signal intensity of Rca1p-HA<sub>3</sub>-binding at the corresponding location. <bold>Target:</bold> orf19 nomenclature according to the <italic>C. albicans</italic> Assembly 19 of Rca1p-HA<sub>3</sub> target gene, based on the location of the locus relative to the binding peak. Absence of information indicates that binding peaks are not clearly associated with promoter of ORFs. If the peak was found in the promoter region common to two adjacent ORFs, the two possible predicted target genes are shown, separated by &#x0201c;and&#x0201d;. <bold>CGD Gene name:</bold> Gene name of the corresponding target gene according to the <italic>Candida</italic> Genome Database (CGD) (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.candidagenome.org\">www.candidagenome.org</ext-link>). <bold>Description:</bold> Gene description according to CGD.</p><p>(DOCX)</p></caption><media xlink:href=\"ppat.1002485.s010.docx\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s011\"><label>Table S2</label><caption><p>\\n<bold>Rca1p-HA<sub>3</sub> binding in 5.5&#x00025; CO<sub>2</sub> dataset.</bold> The following criteria were used: Log<sub>2</sub> pseudo-median signal intensity threshold of &#x02265;0.5 and p-value cut-off of &#x02264;0.01 <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. <bold>Contig19#:</bold> The Contig19 number on which a given binding peak is detected using the Tilescope software <xref rid=\"ppat.1002485-Mogensen1\" ref-type=\"bibr\">[12]</xref>. <bold>Location:</bold> Position of the binding peak in the corresponding Contig19 DNA sequence. <bold>Log2 pseudo-median signal intensity:</bold> Log<sub>2</sub>-transformed pseudo-median signal intensity of Rca1p-HA<sub>3</sub>-binding at the corresponding location. <bold>Target:</bold> orf19 nomenclature according to the <italic>C. albicans</italic> Assembly 19 of Rca1p-HA<sub>3</sub> target gene, based on the location of the locus relative to the binding peak. Absence of information indicates that binding peaks are not clearly associated with promoter of ORFs. If the peak was found in the promoter region common to two adjacent ORFs, the two possible predicted target genes are shown, separated by &#x0201c;and&#x0201d;. <bold>CGD Gene name:</bold> Gene name of the corresponding target gene according to the <italic>Candida</italic> Genome Database (CGD) (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.candidagenome.org\">www.candidagenome.org</ext-link>). <bold>Description:</bold> Gene description according to CGD.</p><p>(DOCX)</p></caption><media xlink:href=\"ppat.1002485.s011.docx\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s012\"><label>Table S3</label><caption><p>\\n<bold>Plasmids used and construct in this study.</bold>\\n</p><p>(DOCX)</p></caption><media xlink:href=\"ppat.1002485.s012.docx\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s013\"><label>Table S4</label><caption><p>\\n<bold>Strains used and constructed in this study.</bold>\\n</p><p>(DOCX)</p></caption><media xlink:href=\"ppat.1002485.s013.docx\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material><supplementary-material content-type=\"local-data\" id=\"ppat.1002485.s014\"><label>Table S5</label><caption><p>\\n<bold>Primers used in this study.</bold>\\n</p><p>(DOCX)</p></caption><media xlink:href=\"ppat.1002485.s014.docx\"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material></sec></body><back><ack><p>We would like to thank Dr. C. Gourlay and Dr. T. von der Haar for their fruitful discussion and their help with <italic>S. cerevisiae</italic> experiments and Dr. Y. Wang for access to equipment. We thank Laura Kemp, Marianna Murphy, Hana &#x0017d;darsk&#x000e1; and Sandra Weber for their technical assistance.</p><p>We are grateful to the School of Biosciences, University of Kent for continuous support.</p></ack><fn-group><fn fn-type=\"COI-statement\"><p>The authors have declared that no competing interests exist.</p></fn><fn fn-type=\"financial-disclosure\"><p>This work was funded by the BBSRC, and the MRC (all FAM), WL was the recipient of a postgraduate training grant from the Royal Thai Fellowship Program, by the Ministry of Education of the Czech Republic LC531, MSM0021620858 and AV0Z50200510 and Howard Hughes Medical Institute International Research Award to ZP. MR was funded by a grant from the Canadian Institutes of Health Research (MOP-84341), and MCL was supported by NIH award R21AI071134. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></fn></fn-group><ref-list><title>References</title><ref id=\"ppat.1002485-Gaugler1\"><label>1</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Gaugler</surname><given-names>CD</given-names></name><name><surname>Lebeck</surname><given-names>L</given-names></name><name><surname>Nakagaki</surname><given-names>B</given-names></name><name><surname>Boush</surname><given-names>GM</given-names></name></person-group>\\n<year>1980</year>\\n<article-title>Orientation of the entomogenous nematode Neoaplectana carpocapsae to carbon dioxide.</article-title>\\n<source>Environ Entomol</source>\\n<volume>9</volume>\\n<fpage>649</fpage>\\n<lpage>652</lpage>\\n</element-citation></ref><ref id=\"ppat.1002485-Bretscher1\"><label>2</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Bretscher</surname><given-names>AJ</given-names></name><name><surname>Busch</surname><given-names>KE</given-names></name><name><surname>de Bono</surname><given-names>M</given-names></name></person-group>\\n<year>2008</year>\\n<article-title>A carbon dioxide avoidance behavior is integrated with responses to ambient oxygen and food in Caenorhabditis elegans.</article-title>\\n<source>Proc Natl Acad Sci U S A</source>\\n<volume>105</volume>\\n<fpage>8044</fpage>\\n<lpage>8049</lpage>\\n<pub-id pub-id-type=\"pmid\">18524954</pub-id></element-citation></ref><ref id=\"ppat.1002485-Suh1\"><label>3</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Suh</surname><given-names>GS</given-names></name><name><surname>Wong</surname><given-names>AM</given-names></name><name><surname>Hergarden</surname><given-names>AC</given-names></name><name><surname>Wang</surname><given-names>JW</given-names></name><name><surname>Simon</surname><given-names>AF</given-names></name><etal/></person-group>\\n<year>2004</year>\\n<article-title>A single population of olfactory sensory neurons mediates an innate avoidance behaviour in Drosophila.</article-title>\\n<source>Nature</source>\\n<volume>431</volume>\\n<fpage>854</fpage>\\n<lpage>859</lpage>\\n<pub-id pub-id-type=\"pmid\">15372051</pub-id></element-citation></ref><ref id=\"ppat.1002485-Jones1\"><label>4</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Jones</surname><given-names>WD</given-names></name><name><surname>Cayirlioglu</surname><given-names>P</given-names></name><name><surname>Kadow</surname><given-names>IG</given-names></name><name><surname>Vosshall</surname><given-names>LB</given-names></name></person-group>\\n<year>2007</year>\\n<article-title>Two chemosensory receptors together mediate carbon dioxide detection in Drosophila.</article-title>\\n<source>Nature</source>\\n<volume>445</volume>\\n<fpage>86</fpage>\\n<lpage>90</lpage>\\n<pub-id pub-id-type=\"pmid\">17167414</pub-id></element-citation></ref><ref id=\"ppat.1002485-Lindskog1\"><label>5</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Lindskog</surname><given-names>S</given-names></name></person-group>\\n<year>1997</year>\\n<article-title>Structure and mechanism of carbonic anhydrase.</article-title>\\n<source>Pharmacol Ther</source>\\n<volume>74</volume>\\n<fpage>1</fpage>\\n<lpage>20</lpage>\\n<pub-id pub-id-type=\"pmid\">9336012</pub-id></element-citation></ref><ref id=\"ppat.1002485-Gotz1\"><label>6</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Gotz</surname><given-names>R</given-names></name><name><surname>Gnann</surname><given-names>A</given-names></name><name><surname>Zimmermann</surname><given-names>FK</given-names></name></person-group>\\n<year>1999</year>\\n<article-title>Deletion of the carbonic anhydrase-like gene NCE103 of the yeast Saccharomyces cerevisiae causes an oxygen-sensitive growth defect.</article-title>\\n<source>Yeast</source>\\n<volume>15</volume>\\n<fpage>855</fpage>\\n<lpage>864</lpage>\\n<pub-id pub-id-type=\"pmid\">10407265</pub-id></element-citation></ref><ref id=\"ppat.1002485-Klengel1\"><label>7</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Klengel</surname><given-names>T</given-names></name><name><surname>Liang</surname><given-names>WJ</given-names></name><name><surname>Chaloupka</surname><given-names>J</given-names></name><name><surname>Ruoff</surname><given-names>C</given-names></name><name><surname>Schroppel</surname><given-names>K</given-names></name><etal/></person-group>\\n<year>2005</year>\\n<article-title>Fungal adenylyl cyclase integrates CO2 sensing with cAMP signaling and virulence.</article-title>\\n<source>Curr Biol</source>\\n<volume>15</volume>\\n<fpage>2021</fpage>\\n<lpage>2026</lpage>\\n<pub-id pub-id-type=\"pmid\">16303561</pub-id></element-citation></ref><ref id=\"ppat.1002485-Elleuche1\"><label>8</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Elleuche</surname><given-names>S</given-names></name><name><surname>Poggeler</surname><given-names>S</given-names></name></person-group>\\n<year>2009</year>\\n<article-title>Beta-carbonic anhydrases play a role in fruiting body development and ascospore germination in the filamentous fungus Sordaria macrospora.</article-title>\\n<source>PLoS One</source>\\n<volume>4</volume>\\n<fpage>e5177</fpage>\\n<pub-id pub-id-type=\"pmid\">19365544</pub-id></element-citation></ref><ref id=\"ppat.1002485-Han1\"><label>9</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>KH</given-names></name><name><surname>Chun</surname><given-names>YH</given-names></name><name><surname>Figueiredo Bde</surname><given-names>C</given-names></name><name><surname>Soriani</surname><given-names>FM</given-names></name><name><surname>Savoldi</surname><given-names>M</given-names></name><etal/></person-group>\\n<year>2010</year>\\n<article-title>The conserved and divergent roles of carbonic anhydrases in the filamentous fungi Aspergillus fumigatus and Aspergillus nidulans.</article-title>\\n<source>Mol Microbiol</source>\\n<volume>75</volume>\\n<fpage>1372</fpage>\\n<lpage>1388</lpage>\\n<pub-id pub-id-type=\"pmid\">20149101</pub-id></element-citation></ref><ref id=\"ppat.1002485-Bahn1\"><label>10</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Bahn</surname><given-names>YS</given-names></name><name><surname>Cox</surname><given-names>GM</given-names></name><name><surname>Perfect</surname><given-names>JR</given-names></name><name><surname>Heitman</surname><given-names>J</given-names></name></person-group>\\n<year>2005</year>\\n<article-title>Carbonic anhydrase and CO2 sensing during Cryptococcus neoformans growth, differentiation, and virulence.</article-title>\\n<source>Curr Biol</source>\\n<volume>15</volume>\\n<fpage>2013</fpage>\\n<lpage>2020</lpage>\\n<pub-id pub-id-type=\"pmid\">16303560</pub-id></element-citation></ref><ref id=\"ppat.1002485-Huang1\"><label>11</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>G</given-names></name><name><surname>Srikantha</surname><given-names>T</given-names></name><name><surname>Sahni</surname><given-names>N</given-names></name><name><surname>Yi</surname><given-names>S</given-names></name><name><surname>Soll</surname><given-names>DR</given-names></name></person-group>\\n<year>2009</year>\\n<article-title>CO(2) regulates white-to-opaque switching in Candida albicans.</article-title>\\n<source>Curr Biol</source>\\n<volume>19</volume>\\n<fpage>330</fpage>\\n<lpage>334</lpage>\\n<pub-id pub-id-type=\"pmid\">19200725</pub-id></element-citation></ref><ref id=\"ppat.1002485-Mogensen1\"><label>12</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Mogensen</surname><given-names>EG</given-names></name><name><surname>Janbon</surname><given-names>G</given-names></name><name><surname>Chaloupka</surname><given-names>J</given-names></name><name><surname>Steegborn</surname><given-names>C</given-names></name><name><surname>Fu</surname><given-names>MS</given-names></name><etal/></person-group>\\n<year>2006</year>\\n<article-title>Cryptococcus neoformans senses CO2 through the carbonic anhydrase Can2 and the adenylyl cyclase Cac1.</article-title>\\n<source>Eukaryot Cell</source>\\n<volume>5</volume>\\n<fpage>103</fpage>\\n<lpage>111</lpage>\\n<pub-id pub-id-type=\"pmid\">16400172</pub-id></element-citation></ref><ref id=\"ppat.1002485-Hall1\"><label>13</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Hall</surname><given-names>RA</given-names></name><name><surname>De Sordi</surname><given-names>L</given-names></name><name><surname>Maccallum</surname><given-names>DM</given-names></name><name><surname>Topal</surname><given-names>H</given-names></name><name><surname>Eaton</surname><given-names>R</given-names></name><etal/></person-group>\\n<year>2010</year>\\n<article-title>CO(2) acts as a signalling molecule in populations of the fungal pathogen Candida albicans.</article-title>\\n<source>PLoS Pathog</source>\\n<volume>6</volume>\\n<fpage>e1001193</fpage>\\n<pub-id pub-id-type=\"pmid\">21124988</pub-id></element-citation></ref><ref id=\"ppat.1002485-Neri1\"><label>14</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Neri</surname><given-names>D</given-names></name><name><surname>Supuran</surname><given-names>CT</given-names></name></person-group>\\n<year>2011</year>\\n<article-title>Interfering with pH regulation in tumours as a therapeutic strategy.</article-title>\\n<source>Nat Rev Drug Discov</source>\\n<volume>10</volume>\\n<fpage>767</fpage>\\n<lpage>77</lpage>\\n<pub-id pub-id-type=\"pmid\">21921921</pub-id></element-citation></ref><ref id=\"ppat.1002485-Wykoff1\"><label>15</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Wykoff</surname><given-names>CC</given-names></name><name><surname>Beasley</surname><given-names>NJ</given-names></name><name><surname>Watson</surname><given-names>PH</given-names></name><name><surname>Turner</surname><given-names>KJ</given-names></name><name><surname>Pastorek</surname><given-names>J</given-names></name><etal/></person-group>\\n<year>2000</year>\\n<article-title>Hypoxia-inducible expression of tumor-associated carbonic anhydrases.</article-title>\\n<source>Cancer Res</source>\\n<volume>60</volume>\\n<fpage>7075</fpage>\\n<lpage>7083</lpage>\\n<pub-id pub-id-type=\"pmid\">11156414</pub-id></element-citation></ref><ref id=\"ppat.1002485-Kovacikova1\"><label>16</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Kovacikova</surname><given-names>G</given-names></name><name><surname>Lin</surname><given-names>W</given-names></name><name><surname>Skorupski</surname><given-names>K</given-names></name></person-group>\\n<year>2010</year>\\n<article-title>The LysR-type virulence activator AphB regulates the expression of genes in Vibrio cholerae in response to low pH and anaerobiosis.</article-title>\\n<source>J Bacteriol</source>\\n<volume>192</volume>\\n<fpage>4181</fpage>\\n<lpage>4191</lpage>\\n<pub-id pub-id-type=\"pmid\">20562308</pub-id></element-citation></ref><ref id=\"ppat.1002485-Aguilera1\"><label>17</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Aguilera</surname><given-names>J</given-names></name><name><surname>Petit</surname><given-names>T</given-names></name><name><surname>de Winde</surname><given-names>JH</given-names></name><name><surname>Pronk</surname><given-names>JT</given-names></name></person-group>\\n<year>2005</year>\\n<article-title>Physiological and genome-wide transcriptional responses of Saccharomyces cerevisiae to high carbon dioxide concentrations.</article-title>\\n<source>FEMS Yeast Res</source>\\n<volume>5</volume>\\n<fpage>579</fpage>\\n<lpage>593</lpage>\\n<pub-id pub-id-type=\"pmid\">15780657</pub-id></element-citation></ref><ref id=\"ppat.1002485-Amoroso1\"><label>18</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Amoroso</surname><given-names>G</given-names></name><name><surname>Morell-Avrahov</surname><given-names>L</given-names></name><name><surname>Muller</surname><given-names>D</given-names></name><name><surname>Klug</surname><given-names>K</given-names></name><name><surname>Sultemeyer</surname><given-names>D</given-names></name></person-group>\\n<year>2005</year>\\n<article-title>The gene NCE103 (YNL036w) from Saccharomyces cerevisiae encodes a functional carbonic anhydrase and its transcription is regulated by the concentration of inorganic carbon in the medium.</article-title>\\n<source>Mol Microbiol</source>\\n<volume>56</volume>\\n<fpage>549</fpage>\\n<lpage>558</lpage>\\n<pub-id pub-id-type=\"pmid\">15813743</pub-id></element-citation></ref><ref id=\"ppat.1002485-Rocha1\"><label>19</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Rocha</surname><given-names>CR</given-names></name><name><surname>Schroppel</surname><given-names>K</given-names></name><name><surname>Harcus</surname><given-names>D</given-names></name><name><surname>Marcil</surname><given-names>A</given-names></name><name><surname>Dignard</surname><given-names>D</given-names></name><etal/></person-group>\\n<year>2001</year>\\n<article-title>Signaling through adenylyl cyclase is essential for hyphal growth and virulence in the pathogenic fungus Candida albicans.</article-title>\\n<source>Mol Biol Cell</source>\\n<volume>12</volume>\\n<fpage>3631</fpage>\\n<lpage>3643</lpage>\\n<pub-id pub-id-type=\"pmid\">11694594</pub-id></element-citation></ref><ref id=\"ppat.1002485-Fonzi1\"><label>20</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Fonzi</surname><given-names>WA</given-names></name><name><surname>Irwin</surname><given-names>MY</given-names></name></person-group>\\n<year>1993</year>\\n<article-title>Isogenic strain construction and gene mapping in Candida albicans.</article-title>\\n<source>Genetics</source>\\n<volume>134</volume>\\n<fpage>717</fpage>\\n<lpage>728</lpage>\\n<pub-id pub-id-type=\"pmid\">8349105</pub-id></element-citation></ref><ref id=\"ppat.1002485-Lorenz1\"><label>21</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Lorenz</surname><given-names>MC</given-names></name><name><surname>Bender</surname><given-names>JA</given-names></name><name><surname>Fink</surname><given-names>GR</given-names></name></person-group>\\n<year>2004</year>\\n<article-title>Transcriptional response of Candida albicans upon internalization by macrophages.</article-title>\\n<source>Eukaryot Cell</source>\\n<volume>3</volume>\\n<fpage>1076</fpage>\\n<lpage>1087</lpage>\\n<pub-id pub-id-type=\"pmid\">15470236</pub-id></element-citation></ref><ref id=\"ppat.1002485-Lorenz2\"><label>22</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Lorenz</surname><given-names>MC</given-names></name><name><surname>Fink</surname><given-names>GR</given-names></name></person-group>\\n<year>2001</year>\\n<article-title>The glyoxylate cycle is required for fungal virulence.</article-title>\\n<source>Nature</source>\\n<volume>412</volume>\\n<fpage>83</fpage>\\n<lpage>86</lpage>\\n<pub-id pub-id-type=\"pmid\">11452311</pub-id></element-citation></ref><ref id=\"ppat.1002485-GarciaGimeno1\"><label>23</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Garcia-Gimeno</surname><given-names>MA</given-names></name><name><surname>Struhl</surname><given-names>K</given-names></name></person-group>\\n<year>2000</year>\\n<article-title>Aca1 and Aca2, ATF/CREB activators in Saccharomyces cerevisiae, are important for carbon source utilization but not the response to stress.</article-title>\\n<source>Mol Cell Biol</source>\\n<volume>20</volume>\\n<fpage>4340</fpage>\\n<lpage>4349</lpage>\\n<pub-id pub-id-type=\"pmid\">10825197</pub-id></element-citation></ref><ref id=\"ppat.1002485-Srikantha1\"><label>24</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Srikantha</surname><given-names>T</given-names></name><name><surname>Borneman</surname><given-names>AR</given-names></name><name><surname>Daniels</surname><given-names>KJ</given-names></name><name><surname>Pujol</surname><given-names>C</given-names></name><name><surname>Wu</surname><given-names>W</given-names></name><etal/></person-group>\\n<year>2006</year>\\n<article-title>TOS9 regulates white-opaque switching in Candida albicans.</article-title>\\n<source>Eukaryot Cell</source>\\n<volume>5</volume>\\n<fpage>1674</fpage>\\n<lpage>1687</lpage>\\n<pub-id pub-id-type=\"pmid\">16950924</pub-id></element-citation></ref><ref id=\"ppat.1002485-Znaidi1\"><label>25</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Znaidi</surname><given-names>S</given-names></name><name><surname>Barker</surname><given-names>KS</given-names></name><name><surname>Weber</surname><given-names>S</given-names></name><name><surname>Alarco</surname><given-names>AM</given-names></name><name><surname>Liu</surname><given-names>TT</given-names></name><etal/></person-group>\\n<year>2009</year>\\n<article-title>Identification of the Candida albicans Cap1p regulon.</article-title>\\n<source>Eukaryot Cell</source>\\n<volume>8</volume>\\n<fpage>806</fpage>\\n<lpage>820</lpage>\\n<pub-id pub-id-type=\"pmid\">19395663</pub-id></element-citation></ref><ref id=\"ppat.1002485-Bates1\"><label>26</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Bates</surname><given-names>S</given-names></name><name><surname>Hughes</surname><given-names>HB</given-names></name><name><surname>Munro</surname><given-names>CA</given-names></name><name><surname>Thomas</surname><given-names>WP</given-names></name><name><surname>MacCallum</surname><given-names>DM</given-names></name><etal/></person-group>\\n<year>2006</year>\\n<article-title>Outer chain N-glycans are required for cell wall integrity and virulence of Candida albicans.</article-title>\\n<source>J Biol Chem</source>\\n<volume>281</volume>\\n<fpage>90</fpage>\\n<lpage>98</lpage>\\n<pub-id pub-id-type=\"pmid\">16263704</pub-id></element-citation></ref><ref id=\"ppat.1002485-Elson1\"><label>27</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Elson</surname><given-names>SL</given-names></name><name><surname>Noble</surname><given-names>SM</given-names></name><name><surname>Solis</surname><given-names>NV</given-names></name><name><surname>Filler</surname><given-names>SG</given-names></name><name><surname>Johnson</surname><given-names>AD</given-names></name></person-group>\\n<year>2009</year>\\n<article-title>An RNA transport system in Candida albicans regulates hyphal morphology and invasive growth.</article-title>\\n<source>PLoS Genet</source>\\n<volume>5</volume>\\n<fpage>e1000664</fpage>\\n<pub-id pub-id-type=\"pmid\">19779551</pub-id></element-citation></ref><ref id=\"ppat.1002485-Vachova1\"><label>28</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Vachova</surname><given-names>L</given-names></name><name><surname>Chernyavskiy</surname><given-names>O</given-names></name><name><surname>Strachotova</surname><given-names>D</given-names></name><name><surname>Bianchini</surname><given-names>P</given-names></name><name><surname>Burdikova</surname><given-names>Z</given-names></name><etal/></person-group>\\n<year>2009</year>\\n<article-title>Architecture of developing multicellular yeast colony: spatio-temporal expression of Ato1p ammonium exporter.</article-title>\\n<source>Environ Microbiol</source>\\n<volume>11</volume>\\n<fpage>1866</fpage>\\n<lpage>77</lpage>\\n<pub-id pub-id-type=\"pmid\">19302539</pub-id></element-citation></ref><ref id=\"ppat.1002485-Schmid1\"><label>29</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Schmid</surname><given-names>A</given-names></name><name><surname>Sutto</surname><given-names>Z</given-names></name><name><surname>Schmid</surname><given-names>N</given-names></name><name><surname>Novak</surname><given-names>L</given-names></name><name><surname>Ivonnet</surname><given-names>P</given-names></name><etal/></person-group>\\n<year>2010</year>\\n<article-title>Decreased soluble adenylyl cyclase activity in cystic fibrosis is related to defective apical bicarbonate exchange and affects ciliary beat frequency regulation.</article-title>\\n<source>J Biol Chem</source>\\n<volume>285</volume>\\n<fpage>29998</fpage>\\n<lpage>30007</lpage>\\n<pub-id pub-id-type=\"pmid\">20639512</pub-id></element-citation></ref><ref id=\"ppat.1002485-Supuran1\"><label>30</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Supuran</surname><given-names>CT</given-names></name></person-group>\\n<year>2008</year>\\n<article-title>Carbonic anhydrases: novel therapeutic applications for inhibitors and activators.</article-title>\\n<source>Nat Rev Drug Discov</source>\\n<volume>7</volume>\\n<fpage>168</fpage>\\n<lpage>181</lpage>\\n<pub-id pub-id-type=\"pmid\">18167490</pub-id></element-citation></ref><ref id=\"ppat.1002485-Schlicker1\"><label>31</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Schlicker</surname><given-names>C</given-names></name><name><surname>Hall</surname><given-names>RA</given-names></name><name><surname>Vullo</surname><given-names>D</given-names></name><name><surname>Middelhaufe</surname><given-names>S</given-names></name><name><surname>Gertz</surname><given-names>M</given-names></name><etal/></person-group>\\n<year>2009</year>\\n<article-title>Structure and inhibition of the CO2-sensing carbonic anhydrase Can2 from the pathogenic fungus Cryptococcus neoformans.</article-title>\\n<source>J Mol Biol</source>\\n<volume>385</volume>\\n<fpage>1207</fpage>\\n<lpage>1220</lpage>\\n<pub-id pub-id-type=\"pmid\">19071134</pub-id></element-citation></ref><ref id=\"ppat.1002485-Schofield1\"><label>32</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Schofield</surname><given-names>CJ</given-names></name><name><surname>Ratcliffe</surname><given-names>PJ</given-names></name></person-group>\\n<year>2005</year>\\n<article-title>Signalling hypoxia by HIF hydroxylases.</article-title>\\n<source>Biochem Biophys Res Commun</source>\\n<volume>338</volume>\\n<fpage>617</fpage>\\n<lpage>626</lpage>\\n<pub-id pub-id-type=\"pmid\">16139242</pub-id></element-citation></ref><ref id=\"ppat.1002485-Parks1\"><label>33</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Parks</surname><given-names>SK</given-names></name><name><surname>Chiche</surname><given-names>J</given-names></name><name><surname>Pouyssegur</surname><given-names>J</given-names></name></person-group>\\n<year>2011</year>\\n<article-title>pH control mechanisms of tumor survival and growth.</article-title>\\n<source>J Cell Physiol</source>\\n<volume>226</volume>\\n<fpage>299</fpage>\\n<lpage>308</lpage>\\n<pub-id pub-id-type=\"pmid\">20857482</pub-id></element-citation></ref><ref id=\"ppat.1002485-Setiadi1\"><label>34</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Setiadi</surname><given-names>ER</given-names></name><name><surname>Doedt</surname><given-names>T</given-names></name><name><surname>Cottier</surname><given-names>F</given-names></name><name><surname>Noffz</surname><given-names>C</given-names></name><name><surname>Ernst</surname><given-names>JF</given-names></name></person-group>\\n<year>2006</year>\\n<article-title>Transcriptional response of Candida albicans to hypoxia: linkage of oxygen sensing and Efg1p-regulatory networks.</article-title>\\n<source>J Mol Biol</source>\\n<volume>361</volume>\\n<fpage>399</fpage>\\n<lpage>411</lpage>\\n<pub-id pub-id-type=\"pmid\">16854431</pub-id></element-citation></ref><ref id=\"ppat.1002485-Carlson1\"><label>35</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Carlson</surname><given-names>JM</given-names></name><name><surname>Chakravarty</surname><given-names>A</given-names></name><name><surname>DeZiel</surname><given-names>CE</given-names></name><name><surname>Gross</surname><given-names>RH</given-names></name></person-group>\\n<year>2007</year>\\n<article-title>SCOPE: a web server for practical de novo motif discovery.</article-title>\\n<source>Nucleic Acids Res</source>\\n<volume>35</volume>\\n<fpage>W259</fpage>\\n<lpage>264</lpage>\\n<pub-id pub-id-type=\"pmid\">17485471</pub-id></element-citation></ref><ref id=\"ppat.1002485-Taylor1\"><label>36</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Taylor</surname><given-names>JW</given-names></name><name><surname>Berbee</surname><given-names>ML</given-names></name></person-group>\\n<year>2006</year>\\n<article-title>Dating divergences in the Fungal Tree of Life: review and new analyses.</article-title>\\n<source>Mycologia</source>\\n<volume>98</volume>\\n<fpage>838</fpage>\\n<lpage>849</lpage>\\n<pub-id pub-id-type=\"pmid\">17486961</pub-id></element-citation></ref><ref id=\"ppat.1002485-Lavoie1\"><label>37</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Lavoie</surname><given-names>H</given-names></name><name><surname>Hogues</surname><given-names>H</given-names></name><name><surname>Mallick</surname><given-names>J</given-names></name><name><surname>Sellam</surname><given-names>A</given-names></name><name><surname>Nantel</surname><given-names>A</given-names></name></person-group>\\n<year>2010</year>\\n<article-title>Evolutionary tinkering with conserved components of a transcriptional regulatory network.</article-title>\\n<source>PLoS Biol</source>\\n<volume>8</volume>\\n<fpage>e1000329</fpage>\\n<pub-id pub-id-type=\"pmid\">20231876</pub-id></element-citation></ref><ref id=\"ppat.1002485-Kaluz1\"><label>38</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Kaluz</surname><given-names>S</given-names></name><name><surname>Kaluzova</surname><given-names>M</given-names></name><name><surname>Liao</surname><given-names>SY</given-names></name><name><surname>Lerman</surname><given-names>M</given-names></name><name><surname>Stanbridge</surname><given-names>EJ</given-names></name></person-group>\\n<year>2009</year>\\n<article-title>Transcriptional control of the tumor- and hypoxia-marker carbonic anhydrase 9: A one transcription factor (HIF-1) show?</article-title>\\n<source>Biochim Biophys Acta</source>\\n<volume>1795</volume>\\n<fpage>162</fpage>\\n<lpage>172</lpage>\\n<pub-id pub-id-type=\"pmid\">19344680</pub-id></element-citation></ref><ref id=\"ppat.1002485-Brennan1\"><label>39</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Brennan</surname><given-names>DJ</given-names></name><name><surname>Jirstrom</surname><given-names>K</given-names></name><name><surname>Kronblad</surname><given-names>A</given-names></name><name><surname>Millikan</surname><given-names>RC</given-names></name><name><surname>Landberg</surname><given-names>G</given-names></name><etal/></person-group>\\n<year>2006</year>\\n<article-title>CA IX is an independent prognostic marker in premenopausal breast cancer patients with one to three positive lymph nodes and a putative marker of radiation resistance.</article-title>\\n<source>Clin Cancer Res</source>\\n<volume>12</volume>\\n<fpage>6421</fpage>\\n<lpage>6431</lpage>\\n<pub-id pub-id-type=\"pmid\">17085655</pub-id></element-citation></ref><ref id=\"ppat.1002485-Demirci1\"><label>40</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Demirci</surname><given-names>A</given-names></name><name><surname>Pometto</surname><given-names>AL</given-names><suffix>3rd</suffix></name><name><surname>Ho</surname><given-names>KL</given-names></name></person-group>\\n<year>1997</year>\\n<article-title>Ethanol production by Saccharomyces cerevisiae in biofilm reactors.</article-title>\\n<source>J Ind Microbiol Biotechnol</source>\\n<volume>19</volume>\\n<fpage>299</fpage>\\n<lpage>304</lpage>\\n<pub-id pub-id-type=\"pmid\">9439005</pub-id></element-citation></ref><ref id=\"ppat.1002485-Zara1\"><label>41</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Zara</surname><given-names>S</given-names></name><name><surname>Gross</surname><given-names>MK</given-names></name><name><surname>Zara</surname><given-names>G</given-names></name><name><surname>Budroni</surname><given-names>M</given-names></name><name><surname>Bakalinsky</surname><given-names>AT</given-names></name></person-group>\\n<year>2010</year>\\n<article-title>Ethanol-independent biofilm formation by a flor wine yeast strain of Saccharomyces cerevisiae.</article-title>\\n<source>Appl Environ Microbiol</source>\\n<volume>76</volume>\\n<fpage>4089</fpage>\\n<lpage>4091</lpage>\\n<pub-id pub-id-type=\"pmid\">20435772</pub-id></element-citation></ref><ref id=\"ppat.1002485-Huang2\"><label>42</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>G</given-names></name><name><surname>Yi</surname><given-names>S</given-names></name><name><surname>Sahni</surname><given-names>N</given-names></name><name><surname>Daniels</surname><given-names>KJ</given-names></name><name><surname>Srikantha</surname><given-names>T</given-names></name><etal/></person-group>\\n<year>2010</year>\\n<article-title>N-acetylglucosamine induces white to opaque switching, a mating prerequisite in Candida albicans.</article-title>\\n<source>PLoS Pathog</source>\\n<volume>6</volume>\\n<fpage>e1000806</fpage>\\n<pub-id pub-id-type=\"pmid\">20300604</pub-id></element-citation></ref><ref id=\"ppat.1002485-Liu1\"><label>43</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>TT</given-names></name><name><surname>Znaidi</surname><given-names>S</given-names></name><name><surname>Barker</surname><given-names>KS</given-names></name><name><surname>Xu</surname><given-names>L</given-names></name><name><surname>Homayouni</surname><given-names>R</given-names></name><etal/></person-group>\\n<year>2007</year>\\n<article-title>Genome-wide expression and location analyses of the Candida albicans Tac1p regulon.</article-title>\\n<source>Eukaryot Cell</source>\\n<volume>6</volume>\\n<fpage>2122</fpage>\\n<lpage>2138</lpage>\\n<pub-id pub-id-type=\"pmid\">17905926</pub-id></element-citation></ref><ref id=\"ppat.1002485-Zhang1\"><label>44</label><element-citation publication-type=\"journal\">\\n<person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>ZD</given-names></name><name><surname>Rozowsky</surname><given-names>J</given-names></name><name><surname>Lam</surname><given-names>HY</given-names></name><name><surname>Du</surname><given-names>J</given-names></name><name><surname>Snyder</surname><given-names>M</given-names></name><etal/></person-group>\\n<year>2007</year>\\n<article-title>Tilescope: online analysis pipeline for high-density tiling microarray data.</article-title>\\n<source>Genome Biol</source>\\n<volume>8</volume>\\n<fpage>R81</fpage>\\n<pub-id pub-id-type=\"pmid\">17501994</pub-id></element-citation></ref></ref-list></back></article>'"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d7175520-1398-42d0-9ed9-db0eddf71e16",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
